	<!DOCTYPE html>
	<html lang="zxx" class="no-js">
	<head>
		<!-- Mobile Specific Meta -->
		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
		<!-- Favicon-->
		<link rel="shortcut icon" href="img/fav.png">
		<!-- Author Meta -->
		<meta name="author" content="codepixer">
		<!-- Meta Description -->
		<meta name="description" content="">
		<!-- Meta Keyword -->
		<meta name="keywords" content="">
		<!-- meta character set -->
		<meta charset="UTF-8">
		<!-- Site Title -->
		<title>Wu Lab @ Columbia | Medicine | Systems Biology </title>

		<link href="https://fonts.googleapis.com/css?family=Poppins:100,200,400,300,500,600,700" rel="stylesheet"> 
			<!--
			CSS
			============================================= -->
			<link rel="stylesheet" href="css/linearicons.css">
			<link rel="stylesheet" href="css/font-awesome.min.css">
			<link rel="stylesheet" href="css/bootstrap.css">
			<link rel="stylesheet" href="css/magnific-popup.css">
			<link rel="stylesheet" href="css/nice-select.css">					
			<link rel="stylesheet" href="css/animate.min.css">
			<link rel="stylesheet" href="css/owl.carousel.css">
			<link rel="stylesheet" href="css/main.css">
		</head>
		<body>

			  <header id="header" id="home">
			    <div class="container">
			    	<div class="row align-items-center justify-content-between d-flex">
				      <div id="logo">
				        <a href="index.html"><img src="img/logo.png" alt="" title="" /></a>
				      </div>
				      <nav id="nav-menu-container">
				        <ul class="nav-menu">
				          <li class="menu-active"><a href="#home">Home</a></li>
				          <li><a href="#research">Research</a></li>
				          <li><a href="#publications">Publications</a></li>
				          <li><a href="#people">People</a></li>
				          <li><a href="#Software">Software</a></li>
						  <li><a href="#protocols">Protocols</a></li>
				          <li><a href="#news">News</a></li>
							<li><a href="#join">Join</a></li>

							<!--
				          <li class="menu-has-children"><a href="">Join</a>
				            <ul>
				              <li><a href="generic.html">Generic</a></li>
				              <li><a href="elements.html">Elements</a></li>
				            </ul>
				          </li> -->
				        </ul>
				      </nav><!-- #nav-menu-container -->		    		
			    	</div>
			    </div>
			  </header><!-- #header -->

			<!-- start banner Area -->
			<section class="banner-area relative" id="home">
				<div class="container" >
					<div class="row fullscreen d-flex align-items-center justify-content-end">
						<div class="banner-content col-lg-7 col-md-12">
							<h4 class="text-uppercase"></h4>
							<h2> <font color="#7F97FF">Wu Lab @ Columbia </font>  </h2>
							<h3 ><font color="#767679"> Medicine </font> | <font color="#0DD46A"> Systems Biology </font> </h3>
							<p>
							<font size="4">
								We study the mechanisms of mammalian mRNA regulation by using integrated experimental and computational approaches. We are also exploring the therapeutic potential of CRISPR-based mRNA-targeting for treating human diseases.</font>
							 
							</p>
						 <!--	<a href="#research" class="primary-btn header-btn text-uppercase">Learn more</a>  -->
							<br><br><br><br><br><br><br><br><br>
						</div>												
					</div>
				</div>
			</section>
			<!-- End banner Area -->	
			
			
			<!-- Start feature Area --> 
			<section class="feature-area section-gap" id="research">
				<div class="container">
					<h2> RESEARCH </h2> <br> <br>
					<div class="row">
						
						<div class="single-feature col-lg-4">
							<img class="img-fluid" src="img/f1.png" alt="">
						  <h4>RNA-centric gene regulation</h4>
						  <p>We study mechanisms of gene regulation, focusing on mRNAs in mammalian systems.  Our current efforts include: 1) noncoding translation in health and disease; 2) functions and mechanisms of mRNA structures. </p>
						  <hr>
							<strong>1</strong>) A widespread role of RNA structures in controlling mRNA 3' end processing and stability.<br>
                        See <a href="http://www.cell.com/cell/fulltext/S0092-8674(17)30490-7">Wu & Bartel, 2017 <em>Cell</em></a><br>			
				          <strong>2</strong>)  A model of how noncoding transcription shapes genome evolution.<br>
				          See <a href="http://www.cell.com/abstract/S0092-8674(13)01406-2">Wu & Sharp, 2013 <em>Cell</em></a><br>
							
						  <strong>	 3</strong>) A mechanism of how RNA processing signals  suppress pervasive transcription and control promoter directionality.<br>
					      See <a href="http://www.nature.com/nature/journal/v499/n7458/full/nature12349.html">Almada & Wu et al. 2013 <em>Nature</em></a> <br>
                          			</div>
						<div class="single-feature col-lg-4">
							<img class="img-fluid" src="img/f2.png" alt="">
							<h4>CRISPR technology & therapy</h4>
							<p>We characterize CRISPR technologies using unbiased genomic assays. We are also exploring the therapeutic potential of CRISPR/Cas13-based mRNA targeting therapeutics for human diseases.<br>
							</p>
							<hr>
					        <strong>1</strong>)  The first genome-wide analysis of Cas9 targeting in mammalian cells.<br>
					        See 
						      <a href="http://www.nature.com/nbt/journal/v32/n7/full/nbt.2889.html">Wu et al. 2014 <em>Nature Biotechnology</em></a><br>
				          <strong>2</strong>) CRISPR/Cas-based technologies for editing the genome, epigenome, and transcriptome.<br>
			            See <a href="http://www.sciencemag.org/content/339/6121/819.short">Cong et al. 2013 <em>Science</em></a>, <a href="http://www.nature.com/nature/journal/v520/n7546/full/nature14299.html">Ran et al. 2015 <em>Nature</em></a>, <a href="http://www.cell.com/cell/fulltext/S0092-8674(16)31153-9">Liu et al. 2016 <em>Cell</em></a>, <a href="http://www.cell.com/cell/fulltext/S0092-8674(18)30049-7">Liu et al. 2018 <em>Cell</em></a>, <a href="">Zhang et al. 2018 <em>Cell</em></a>						</div>
						<div class="single-feature col-lg-4">
							<img class="img-fluid" src="img/f3.png" alt="">
							<h4>Computational biology</h4>
						  <p>We develop new quantitative models, machine learning approaches, and computational tools to understand RNA biology and improve CRISPR guide RNA design.  </p>
							<hr>
							<p><strong>						    1</strong>) A  web server called <a href="http://kplogo.wi.mit.edu/" target="_blank">kpLogo</a> for position-specific short motif analysis.<br>
							  See
							  <a href="http://dx.doi.org/10.1093/nar/gkx323">Wu & Bartel 2017 <em>Nucleic Acids Research</em></a><br>
						      <strong>2</strong>) Network-based global prediction of human disease genes.<br>
						      See
						      <a href="http://msb.embopress.org/content/4/1/189">Wu et al. 2008 <em>Molecular Systems Biology</em></a><br>
				          <strong>3</strong>) Network alignment-based identification of disease families and associated gene modules.<br>
				          See
				          <a href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/25/1/98">Wu et al. 2009 <em>Bioinformatics</em></a></p>
						</div>												
					</div>
				</div>	
			</section>
			<!-- End feature Area -->


			<!-- Start publication Area -->
			<section class="offered-area" id="publications">
				<div class="container">
					<div style="margin-left:40px;text-align:left">
</div>
<div>
<div>
	<br><br>
	<h2> PUBLICATIONS </h2>
	
<div><span style="color:rgb(0,0,255)"><font size="3"><a href="http://scholar.google.com/citations?user=qxEYZ9UAAAAJ"><img src="http://scholar.google.co.uk/intl/en/scholar/feather-72.png" style="display:inline;margin:5px 10px 0px 0px;float:left" width="30" height="60"></a><br>
<a href="http://scholar.google.com/citations?user=qxEYZ9UAAAAJ">Google Scholar Citations<br>
</a></font></span></div>
<span style="font-size:x-large;line-height:34.9090919494629px;background-color:transparent"><span style="font-size:x-large;line-height:34.9090919494629px;background-color:transparent"><span style="font-size:x-large;line-height:34.9090919494629px;background-color:transparent"><br>
</span></span></span></div>
	
	co-first author<sup>*</sup>,
	co-corresponding author<sup>^</sup> <br><br>
	
	<p><font size="5"><strong><strong>2018</strong></strong></font></p>
	<hr>

	<p> Cheng Zhang, Silvana Konermann, Nicholas J. Brideau, Peter Lotfy, <u>Xuebing Wu</u>, Scott J. Novick, Timothy Strutzenberg, Patrick R. Griffin, Patrick D. Hsu, Dmitry Lyumkis<br>
  <a href="" target="https://www.sciencedirect.com/science/article/pii/S0092867418311735">Structural basis for the RNA-guided ribonuclease activity of CRISPR-Cas13d.</a><br>
  <strong><strong><em>Cell</em></strong></strong>, 2018, 175:212-223</p>
	
  <p>Josh Tycko, Luis A Barrera, Nicholas Huston, Ari E Friedland, <u>Xuebing Wu</u>, Jonathan S Gootenberg, Omar O Abudayyeh, Vic E Myer, Christopher J Wilson, Patrick D Hsu<br>
  <a href="https://www.nature.com/articles/s41467-018-05391-2" target="_blank">Pairwise library screen systematically interrogates Staphylococcus aureus Cas9 specificity in human cells.</a><br>
  <strong><strong><em>Nature Communications</em></strong></strong>, 2018, 9:2962</p>
	
  <p>Wei Jiang*, Yuehua Wei*, Yong Long*, Arthur Owen, Bingying Wang, <u>Xuebing Wu</u>, Shuo Luo, Yongjun Dang, Dengke K. Ma<br>
  <a href="https://elifesciences.org/articles/35037" target="_blank">A genetic program mediates cold-warming response and promotes stress-induced phenoptosis in C. elegans</a><br>
  <strong><strong><em>eLife</em></strong></strong>, 2018, 7:e35037</p>
	
  <p>X Shawn Liu, Hao Wu, Marine Krzisch, <u>Xuebing Wu</u>, John Graef, Julien Muffat, Denes Hnisz, Charles H. Li, Bingbing Yuan, Chuanyun Xu, Yun Li, Dan Vershkov, Angela Cacace, Richard A. Young, 
    Rudolf Jaenisch<br>
  <a href="http://www.cell.com/cell/fulltext/S0092-8674(18)30049-7" target="_blank">Rescue of fragile X syndrome neurons by DNA methylation editing of the FMR1 gene</a><br>
  <em><strong><strong>Cell</strong></strong></em>, 2018, 172:979–992<br></p>
	
  <p>Anthony C. Chiu*, Hiroshi I. Suzuki*, <u>Xuebing Wu</u>, Dig B. Mahat, Andrea J. Kriz, Phillip A. Sharp<br>
  <a href="http://www.cell.com/molecular-cell/fulltext/S1097-2765(18)30006-6" target="_blank">Transcriptional Pause Sites Delineate Stable Nucleosome-Associated Premature Polyadenylation Suppressed by U1 snRNP</a> <br>
  <strong><strong><em>Molecular Cell</em></strong></strong>, 2018, 69:648–663<br></p>
	
  	<p><font size="5"><strong><strong>2017</strong></strong></font></p>
  	<hr>

	  
<p><u>Xuebing Wu</u>, David P. Bartel<br>
<a href="http://www.cell.com/cell/fulltext/S0092-8674(17)30490-7" target="_blank">Widespread influence of 3′-end structures on mammalian mRNA processing and stability</a><br>
<strong><strong><em>Cell</em></strong></strong>, 2017, 169: 905–917<br></p>
	
<p><u>Xuebing Wu</u>^, David P. Bartel^ (^co-corresponding author)<br>
  <a href="http://dx.doi.org/10.1093/nar/gkx323" target="_blank"><em>k</em>pLogo: positional <em>k</em>-mer analysis reveals hidden specificity in biological sequences</a><br>
  <strong><strong><em>Nucleic Acids Research</em></strong></strong>, <a href="https://doi.org/10.1093/nar/gkx323" target="_blank">gkx323</a> <br>
  <a href="http://kplogo.wi.mit.edu">http://kplogo.wi.mit.edu</a></p>
	
	<p><font size="5"><strong><strong>2016</strong></strong></font></p>
	<hr>
	
<p>X Shawn Liu*, Hao Wu*, Xiong Ji, Yonatan Stelzer, <u>Xuebing Wu</u>, Szymon Czauderna, Jian Shu, Chikdu S. Shivalila, Daniel Dadon, Richard A. Young, Rudolf Jaenisch<br>
 <a href="http://www.cell.com/cell/fulltext/S0092-8674(16)31153-9" target="_blank">Editing DNA methylation in the mammalian genome</a><br>
  <strong><strong><em>Cell</em></strong></strong>, 2016, 167:233-247<br>
	
<em style="background-color:transparent;font-size:13.3333px"><span style="color:rgb(255,0,0)"><span>&nbsp;&nbsp; &nbsp;</span> Journal highlight: <a href="http://www.nature.com/nature/journal/v537/n7622/full/537588c.html" target="_blank">Nature</a></span>&nbsp;</em></p>

<p>Xiaochang Zhang, Ming Hui Chen, <u>Xuebing Wu</u>, Andrew Kodani, Jean Fan, Ryan Doan, Manabu Ozawa, Jacqueline Ma, Nobuaki Yoshida, Jeremy F Reiter, Douglas L. Black, Peter V Kharchenko, Phillip A. Sharp, Christopher A. Walsh<br>
<a href="http://www.cell.com/cell/fulltext/S0092-8674(16)30932-1" target="_blank">Cell type-specific alternative splicing governs cell fate in the developing cerebral cortex</a><br>
<strong><strong><em>Cell</em></strong></strong>, 2016, 166:1147-1162</p>
	
<br>
	
	<p><font size="5"><strong><strong>2015</strong></strong></font></p>
	<hr>
	
<p>F. Ann Ran*, Le Cong*, Winston X. Yan*, David A. Scott, Jonathan S. Gootenberg, Andrea J. Kriz, Bernd Zetsche, Ophir Shalem, <u>Xuebing Wu</u>, Kira S. Makarova, Eugene V. Koonin, Phillip A. Sharp, Feng Zhang<br>
<a href="http://www.nature.com/nature/journal/v520/n7546/full/nature14299.html" target="_blank">In vivo genome editing using Staphylococcus aureus Cas9</a><br>
<em><strong><strong>Nature</strong></strong></em>, 2015, 520:186-191</p>
	
	<p><font size="5"><strong><strong>2014</strong></strong></font></p>
	<hr>
	
<p><u>Xuebing Wu</u>, Andrea J. Kriz, Phillip A. Sharp <br>
  <a href="http://link.springer.com/article/10.1007/s40484-014-0030-x" target="_blank">Target specificity of the CRISPR-Cas9 system</a></strong></strong><br><em><strong><strong>Quantitative Biology</strong></strong></em>, 2014, 2:59-70<br>
  </p>
			
<p><u>Xuebing Wu</u>, David A. Scott, Andrea J. Kriz, Anthony C. Chiu, Patrick D. Hsu, Daniel B. 
Dadon, Albert W. Cheng, Alexandro E. Trevino, Silvana Konermann, Sidi Chen, Rudolf Jaenisch, Feng Zhang, Phillip A. Sharp<br>
<a href="http://www.nature.com/nbt/journal/v32/n7/full/nbt.2889.html" target="_blank">Genome-wide binding of the CRISPR endonuclease Cas9 in mammalian cells</a><br>

	<em><strong><strong>Nature Biotechnology</strong></strong></em>, 2014, 32:670-676<br></p>
			
<p>Sidi Chen, Yuan Xue, <u>Xuebing Wu</u>, Le Cong, Arjun Bhutkar, Eric Bell, Feng Zhang, Robert Langer, Phillip A. Sharp<br>
<a href="http://genesdev.cshlp.org/content/28/10/1054" target="_blank">Global microRNA depletion suppresses tumor angiogenesis</a><br>
<strong><strong><em>Genes &amp; Development</strong></strong></em>, 2014, 28:1054-1067 <br>
</p>
			
	<p><font size="5"><strong><strong>2013</strong></strong></font></p>
	<hr>
			
<p><u>Xuebing Wu</u>, Phillip A. Sharp<br>
<a href="http://www.cell.com/abstract/S0092-8674(13)01406-2" target="_blank">Divergent transcription: a driving force for new gene origination?</a><br>
<em><strong><strong>Cell</strong></strong></em>, 2013, 155:990-996<br>			
<em><span style="color:rgb(255,0,0)">"Leading Edge Featured Article" <span style="color:rgb(61,133,198)">in Cell</span></span></em>
</p>
			
<p>Albert E. Almada*, <u>Xuebing Wu</u>*, Andrea J. Kriz, Christopher B. Burge, Phillip A. Sharp (*equal contribution)<br>
<a href="http://www.nature.com/nature/journal/v499/n7458/full/nature12349.html" target="_blank">Promoter directionality is controlled by U1 snRNP and polyadenylation signals</a><br>
<strong><strong><em>Nature</em></strong></strong>, 2013, 499:360–363<br>

<a href="https://f1000.com/prime/718098558?bd=1" target="_blank"><img alt="Access the recommendation on F1000Prime" src="https://s3.amazonaws.com/cdn.f1000.com/images/badges/badgef1000.gif"></a> <br>

<em><span style="color:rgb(255,0,0)"> &nbsp;&nbsp;&nbsp; Journal highlight: <a href="http://www.nature.com/nrg/journal/v14/n8/full/nrg3544.html" target="_blank"><span style="color:rgb(61,133,198)">Nature Reviews Genetics</span></a></span>  |  </em><a href="https://www.nature.com/articles/cr2013100" target="_blank"><span style="color:rgb(61,133,198)"><em>Cell Research</em></span></a>  |  <a href="http://www.nature.com/nsmb/journal/v20/n9/full/nsmb.2657.html" target="_blank"><span style="color:rgb(61,133,198)"><em>Nature Structural Molecular Biology</em></span></a>
<em><br>
<span style="color:rgb(255,0,0)"><span>  &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp;  Media</span></span>:</em><em><em> </em></em><em><span style="color:rgb(255,0,0)"><a href="http://news.mit.edu/2013/reading-dna-backward-and-forward-0623" target="_blank"><span style="color:rgb(61,133,198)">MIT News</span></a></span>  |</em> <em><a href="http://www.sciencedaily.com/releases/2013/06/130624141412.htm" target="_blank"><span style="color:rgb(61,133,198)">ScienceDaily</span></a>  |  </em><em><a href="http://www.csmonitor.com/Science/2013/0625/Don-t-touch-my-junk-DNA%21-says-gene-signal-sequence" target="_blank"><span style="color:rgb(61,133,198)">The Christian Science Monitor</span></a>  |&nbsp; </em><em><a href="http://www.genengnews.com/keywordsandtools/print/4/31804/" target="_blank"><span style="color:rgb(61,133,198)">Genetic Engineering &amp; Technology News</span></a></em><br>
</p>
			
<p> Le Cong, F Ann Ran, David Cox, Shuailiang Lin, Robert Barretto, Naomi Habib, Patrick D Hsu, <u>Xuebing Wu</u>, Wenyan Jiang, Luciano A Marraffini, Feng Zhang<br>
<a href="http://www.sciencemag.org/content/339/6121/819.short" target="_blank">Multiplex genome engineering using CRISPR/Cas systems</a><strong><strong><em><br>
Science</em></strong></strong>, 2013, 339 (6121):819-823. <br>
			
	<em><span style="color:rgb(255,0,0)"><span>&nbsp;&nbsp; &nbsp;</span> highlighted</span> in <span style="color:rgb(61,133,198)"><span style="background-color:rgb(255,255,255)"><a href="http://www.sciencemag.org/content/339/6121/768.full" target="_blank"><span style="color:rgb(61,133,198)">Science</span></a>&nbsp; <span style="color:rgb(0,0,0)">|</span>&nbsp; </span></span><span style="color:rgb(61,133,198)"><span style="background-color:rgb(255,255,255)"><span style="color:rgb(61,133,198)"><span style="background-color:rgb(255,255,255)"><a href="http://www.nature.com/nrg/journal/v14/n2/full/nrg3409.html" target="_blank"><span style="color:rgb(61,133,198)">Nature Reviews Genetics</span></a></span></span>&nbsp; <span style="color:rgb(0,0,0)">|</span></span></span><span style="color:rgb(61,133,198)"><span style="color:rgb(0,0,0)">&nbsp;</span> </span><span style="color:rgb(61,133,198)"><span style="color:rgb(61,133,198)"><span style="background-color:rgb(255,255,255)"><span style="color:rgb(61,133,198)"><span style="background-color:rgb(255,255,255)"><a href="http://www.nature.com/nmeth/journal/v10/n3/full/nmeth.2389.html" target="_blank"><span style="color:rgb(61,133,198)">Nature Methods</span></a></em><br></em></em>
</p>
	
<p>Patrick D. Hsu, David A. Scott, Joshua A. Weinstein, F. Ann Ran, Silvana Konermann, Vineeta Agarwala, Yinqing Li, Eli J. Fine, <u>Xuebing Wu</u>, Ophir Shalem, Thomas J. Cradick, Luciano A. Marraffini, Gang Bao, and Feng Zhang<br>
<a href="http://www.nature.com/nbt/journal/v31/n9/full/nbt.2647.html" target="_blank">DNA Targeting Specificity of the RNA-guided Cas9 Nuclease</a><strong><strong><em><br>
Nature Biotechnology</em></strong></strong>, 2013, 31:827-832<br>
<br></p>
	
<p>Anna Lyubimova, Shalev Itzkovitz, Jan Philipp Junker, Zi Peng Fan,  <u>Xuebing Wu</u>, Alexander van Oudenaarden<br>
<a href="http://www.nature.com/nprot/journal/v8/n9/full/nprot.2013.109.html" target="_blank">Single-molecule mRNA detection and counting in mammalian tissue</a><strong><strong><em><br>
Nature Protocols</em></strong></strong>, 2013, 8:1743–1758 <a href="https://github.com/xuebingwu/ImageM" target="_blank">(download software)</a></p>
	
<p>Yong Chen, <u>Xuebing Wu</u>, Rui Jiang<br>
<a href="http://www.biomedcentral.com/1755-8794/6/57/abstract" target="_blank">Integrating human omics data to prioritize candidate genes</a><br>
<strong><strong><em>BMC Medical Genomics</em></strong></strong>, 2013, 6:57<br>
</p>
	
	<p><font size="5"><strong><strong>2010</strong></strong></font></p>
	<hr>
	
<p><u>Xuebing Wu</u>, Shao Li<br>
  <a href="http://www.mit.edu/%7Ewuxbl/csb.pdf" target="_blank">Cancer gene prediction using a network approach</a><br>
  Chapter 11 in: <strong><strong><em>Cancer Systems Biology</em></strong></strong> (Ed. Edwin Wang). Series: Chapman &amp; Hall/CRC Mathematical &amp; Computational Biology, USA: CRC Press 2010. [<a href="http://www.amazon.com/Systems-Biology-Chapman-Mathematical-Computational/dp/1439811857" target="_blank"><span style="color:rgb(61,133,198)">Amazon</span></a>]	    
  </pp>
  
<p><font size="5"><strong><strong>2009</strong></strong></font></p>
<hr>

		
        <p> <u>Xuebing Wu</u>, Qifang Liu, Rui Jiang<br>
              <a href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/25/1/98" target="_blank">Align human interactome with phenome to identify causative genes and networks underlying disease families</a><br>
			<strong><strong><em>Bioinformatics</em></strong></strong>, 2009, 25 (1): 98-104 [<a href="http://bioinfo.au.tsinghua.edu.cn/alignpi/" target="_blank"><span style="color:rgb(61,133,198)">Data</span></a>]<br>
        </p>
			
<p>Wanwan Tang, <u>Xuebing Wu</u>, Rui Jiang, Yanda Li<br>
 <a href="http://www.plosgenetics.org/article/info:doi%2F10.1371%2Fjournal.pgen.1000464" target="_blank">Epistatic module detection for case-control studies: A Bayesian model with a Gibbs sampling strategy</a><br>
 <strong><strong><em>PLoS Genetics</em></strong></strong>, 2009, (5):e1000464<br>
			</p>
			
<p>Rui Jiang, Wanwan Tang, <u>Xuebing Wu</u>, Wenhui Fu<br>
<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2648748" target="_blank">A random forest approach to the detection of epistatic interactions in case-control studies</a><br>
<strong><strong><em>BMC Bioinformatics</em></strong></strong>, 2009, 10 (Suppl 1):S65 <br>
			</p>
			
	<p><font size="5"><strong><strong>2008</strong></strong></font></p>
	<hr>
			
<p> <u>Xuebing Wu</u>, Rui Jiang, Michael Q. Zhang, Shao Li<br>
  <a href="http://msb.embopress.org/content/4/1/189" target="_blank">Network-based global inference of human disease genes</a><br>
  <strong><strong><em>Molecular Systems Biology</em></strong></strong>, 2008, 4:189.
  
</div>
</div>
</div>

			</section>
			<!-- End publication Area -->
			
<!-- Start people Area --> 
			<section class="feature-area section-gap" id="people">
				<div class="container">
									<h2>Lab members</h2>
<br><br>

					<div class="row">
						<table border="0">
							
							
			<tr>
				<td> 2019/07/01 </td>
            <td width="400" height="300" bgcolor="#FFFFFF"><img src="img/lab-20190701.jpg" width="400" height="300" /></td>
				<br><br><br><br>
          </tr>
							
			<tr>
            <td width="200" height="200" bgcolor="#FFFFFF"><img src="img/xuebingwu.jpg" width="180" height="180" /></td>
            <td bgcolor="#FFFFFF"><div class="">
              <h4 align="left">Xuebing Wu</h4>
              <div align="left">
                <p><br>
                  Assistant Professor of <a href="http://www.columbiamedicine.org/" target="new">Medicine</a> and <a href="https://systemsbiology.columbia.edu/" target="new">Systems Biology</a><br>
                xw2629@columbia.edu  (<a href="cv/Xuebing_Wu-CV-2018.pdf">CV</a>) </p>
							<p>Postdoc, David Bartel lab, Whitehead Institute<br>
							   PhD in Computational and Systems Biology, MIT (Phil Sharp and Chris Burge lab)<br>
							   BS/MS in Control Science and Engineering, Tsinghua University, Beijing
				</p>
              </div>
            </div></td>
          </tr>
							
			<tr>
            <td width="200" height="200" bgcolor="#FFFFFF"><img src="img/zihengchen20190201.jpg" width="180" height="180" /></td>
            <td bgcolor="#FFFFFF"><div class="">
              <h4 align="left">Ziheng (Calvin) Chen</h4>
              <div align="left">
		      <br> 
                <p>Master of Science Program, Department of Biomedical Engineering, Columbia University  <br>
		   BSc Biochemistry, University of Liverpool, Liverpool
		      </p>
		      
							<p> zc2418@columbia.edu</p>
							<p>
							</p>
              </div>
            </div></td>
          </tr>
							
													
			<tr>
            <td width="200" height="200" bgcolor="#FFFFFF"><img src="img/alexis_aparicio20190219.jpg" width="180" height="180" /></td>
            <td bgcolor="#FFFFFF"><div class="">
              <h4 align="left">Alexis Aparicio</h4>
              <div align="left">
		      <br> 
                <p>Technician<br>
		   BA in Biochemistry and Molecular Biology, Boston University
		      </p>
		      
							<p> aoa2145@columbia.edu</p>
							<p>
							</p>
              </div>
            </div></td>
          </tr>
			
							
																	
			<tr>
            <td width="200" height="200" bgcolor="#FFFFFF"><img src="img/michaelmurphy20190301.jpg" width="180" height="180" /></td>
            <td bgcolor="#FFFFFF"><div class="">
              <h4 align="left">Michael Murphy</h4>
              <div align="left">
		      <br> 
                <p>Postdoc Research Scientist<br>
			Ph.D. in Molecular, Cellular, and Developmental Biology, City University of New York
		      </p>
		      
							<p> mrm2267@cumc.columbia.edu</p>
							<p>
							</p>
              </div>
            </div></td>
          </tr>
							
										<tr>
            <td width="200" height="200" bgcolor="#FFFFFF"><img src="img/ellenalt20190528.jpg" width="180" height="180" /></td>
            <td bgcolor="#FFFFFF"><div class="">
              <h4 align="left">Ellen Alt</h4>
              <div align="left">
		      <br> 
                <p>Undergraduate student (Cellular and Molecular Biology)<br>
			Barnard College of Columbia University
		      </p>
		      
							<p> era2141@barnard.edu</p>
							<p>
							</p>
              </div>
            </div></td>
          </tr>
							
										<tr>
            <td width="200" height="200" bgcolor="#FFFFFF"><img src="img/ziqiangwang20190601.jpg" width="180" height="180" /></td>
            <td bgcolor="#FFFFFF"><div class="">
              <h4 align="left">Ziqiang Wang</h4>
              <div align="left">
		      <br> 
                <p>Postdoc Research Scientist<br>
			PhD in Cell Biology, Tsinghua University
		      </p>
		      
							<p> email: zw2651@cumc.columbia.edu </p>
							<p>
							</p>
              </div>
            </div></td>
          </tr>
			
									
						</table>
						
						
					</div>
				</div>	
			</section>
			<!-- End feature Area -->
			


			<!-- Start Software Area -->
			<section class="offered-area" id="Software">
				<div class="container">
					<div style="margin-left:40px;text-align:left">
</div>
<div>
<div>
	<br><br>
	<h2> Software </h2>
	
<div>
<div>
<div>
	<div><br><br><font color="#e69138"><font size="4"><strong><strong><a href="https://github.com/xuebingwu" target="new">Lab GitHub</a></strong></strong></font></font><font size="4" color="#e69138"><strong><strong><br><br><br>
</strong></strong></font></div>
	
<div><font color="#e69138"><font size="4"><strong><strong><em>k</em>pLogo</strong></strong></font></font><font size="4" color="#e69138"><strong><strong><br>
</strong></strong></font></div>
<div>
<hr>
</div>
<font size="4" color="#e69138"><span style="line-height:22px"><strong><strong>
<table width="100%">
<tbody>
<tr>
<td><br>
</td>
<td width="80%">
<div><strong><strong>
<table style="display:inline!important" width="100%">
<tbody style="display:inline!important">
<tr style="display:inline!important">
<td style="display:inline!important" width="80%">
<div style="display:inline!important"><font size="2" color="#000000"><span style="font-weight:normal">
A webserver for positional k-mer analysis<strong><strong><br>
</strong></strong><strong><strong>Links:</strong></strong> <a href="http://kplogo.wi.mit.edu" target="_blank">http://kplogo.wi.mit.edu</a>
			
			<br>
			<strong><strong>Citation: </strong></strong></span></font><font size="2" color="#000000"><span style="font-weight:normal"><strong><strong><font size="4" color="#e69138"><span style="line-height:22px"><strong><strong><strong><strong><font size="2" color="#000000"><span style="font-weight:normal">Wu &amp;</span></font></strong></strong></strong></strong></span></font></strong></strong><font size="4" color="#e69138"><span style="line-height:22px"><font size="2" color="#000000"><span style="font-weight:normal"> Bartel (2017)</span></font></span></font><strong><strong><font size="4" color="#e69138"><span style="line-height:22px"><strong><strong><strong><strong><font size="2" color="#000000"><span style="font-weight:normal"> <em><strong><strong>Nucleic Acids Res</strong></strong></em>, <strong><strong>gkx323<br>
</strong></strong></span></font></strong></strong></strong></strong></span></font></strong></strong></span></font></div>
</td>
</tr>
</tbody>
</table>
</strong></strong></div>
</td>
</tr>
</tbody>
</table>
</strong></strong></span></font>
	
<table style="display:inline!important" width="100%">
<tbody style="display:inline!important">
<tr style="display:inline!important">
<td style="display:inline!important" width="80%"><br>
<br>
</td>
</tr>
</tbody>
</table>
</strong></strong></div>
</td>
</tr>
</tbody>
</table>
</strong></strong></div>
</td>
</tr>
</tbody>
</table>
</strong></strong></span></font><font size="4" color="#e69138"><strong><strong>CIPHER</strong></strong></font>
<div>
<hr>
</div>
<div><font size="4" color="#e69138"><span style="line-height:22px"><strong><strong>
<table width="100%">
<tbody>
<tr>
<td><img alt="" src="img/cipher.jpg" style="border:10px solid white" height="158"></td>
<td width="80%">
<div><strong><strong>
<table style="display:inline!important" width="100%">
<tbody style="display:inline!important">
<tr style="display:inline!important">
<td style="display:inline!important" width="80%">
<div style="display:inline!important"><font size="2" color="#000000"><span style="font-weight:normal"><strong><strong>CIPHER</strong></strong>, which stands for Correlating interactome and phenome 
networks to predict disease genes, is a computational framework we 
proposed to prioritize human disease genes. It was one of the first 
studies to explore interactome-phenome wide gene-disease relationships, 
and generated the first comprehensive genetic landscape of human 
disease, connecting 5080 human disease phenotypes with 14433 human genes.
			<br>
<br>
			<strong><strong>Links:</strong></strong> Click here to <a href="http://bioinfo.au.tsinghua.edu.cn/jianglab/cipher/" target="_blank"> Download dataset </a>
			or <a href="http://bioinfo.au.tsinghua.edu.cn/cipher/cipher_search.html" target="_blank"> Explore predictions </a> (try <a href="http://rulai.cshl.edu/tools/cipher/" target="_blank"> mirror at CSHL </a>)
			
			<br>
<br>
			<strong><strong>Citation:</strong></strong> Wu X, Jiang R, Zhang MQ, Li S (2008) Network-based global inference of human disease genes. Molecular Systems Biology, 4:189</span></font><strong><strong>
<table style="display:inline!important" width="100%">
<tbody style="display:inline!important">
<tr style="display:inline!important">
<td style="display:inline!important" width="80%"><br>
</td>
</tr>
</tbody>
</table>
</strong></strong></div>
</td>
</tr>
</tbody>
</table>
</strong></strong></div>
</td>
</tr>
</tbody>
</table>
</strong></strong></span></font></div>
<br>









<div><font size="4" color="#e69138"><strong><strong><br>
</strong></strong></font></div>
<span style="color:rgb(255,153,0)"><font size="4">
AlignPI
</font></span>
<div>
<hr>
<br>
</div>
<div><font size="4" color="#e69138"><span style="line-height:22px"><strong><strong>
<table width="100%">
<tbody>
<tr>
<td><img alt="" src="img/alignpi.gif" style="border:10px solid white" height="158">
</td>
<td width="80%">
<br>
<div><font style="font-weight:normal" size="2" color="#000000"><strong><strong>AlignPI</strong></strong> is the abbreviation of "Align Phenome &amp; Interactome." We have performed the first alignment of human phenome and interactome network, and identified 39 bi-modules and made predictions of candidate genes for 70 diseases. Bi-module consists of two inter-connected modules: a network of diseases and a network of genes.&nbsp; <br>
<br>
<strong><strong>Links</strong></strong>: The </font><font style="font-weight:normal" size="2" color="#000000"><font size="4" color="#e69138"><span style="line-height:22px"><strong><strong><strong><strong><font size="2" color="#000000"><span style="font-weight:normal"><a href="http://bioinfo.au.tsinghua.edu.cn/alignpi/" target="_blank">AlignPI</a></span></font></strong></strong></strong></strong></span></font> website provides a search interface for all identified bi-modules and candidate gene predictions, and corresponding functional enrichment analysis.&nbsp; <br>
<br>
<strong><strong>Citation</strong></strong>: Wu X, Liu Q, Jiang R (2009) Align human interactome with phenome to identify causative genes and networks underlying disease families. Bioinformatics, 25 (1): 98-104</font></div>
</td>
</tr>
</tbody>
</table>
</strong></strong></span></font></div>
<br>
<div><font size="4" color="#e69138"><span style="line-height:22px"><strong><strong>
<div><font size="4" color="#e69138"><span style="line-height:22px"><strong><strong>
<table width="100%">
<tbody>
<tr>
<td><br>
</td>
<td width="80%"><br>
</td>
</tr>
</tbody>
</table>
</strong></strong></span></font></div>
</strong></strong></span></font></div>
</div>
</div>
</div>

				</div>	
			</section>
			<!-- End Software Area -->
			
		
			<!-- Start protocols Area -->
			<section class="offered-area" id="protocols">
				<div class="container">
					<div style="margin-left:40px;text-align:left">
</div>
<div>
<div>
	<br><br>
	<h2> Protocols </h2>
	<br>
	
	<p><a href="protocols/2P-seq-XW.pdf" target="_blank">4sU-2P-Seq and DIM-2P-seq protocol (Mar 2017) (PDF)  </a></p>
	
	<br><br>
</div></div>
				</div>	
			</section>
			<!-- End protocols Area -->

			<!-- Start news Area -->
			<section class="offered-area" id="news">
				<div class="container">
					<div style="margin-left:40px;text-align:left">
</div>
<div>
<div>
	<br><br>
	<h2> Lab news </h2>
	<br>
	
	<p><strong><strong>June 1, 2019</strong></strong> Ziqiang Wang joined the lab as a postdoc research scientist. Welcome Ziqiang!</p>

	<p><strong><strong>May 28, 2019</strong></strong> Ellen Alt joined the lab as a summer research student. Welcome Ellen!</p>

	<p><strong><strong>Mar 1, 2019</strong></strong> Michael Murphy joined the lab as a staff associate. Welcome Michael!</p>

	<p><strong><strong>Feb 19, 2019</strong></strong> Alexis Aparicio joined the lab as our first Technician. Welcome Alex!</p>

	<p><strong><strong>Jan, 2019</strong></strong> Ziheng (Calvin) Chen, a graduate student in the BME Master of Science program joined the lab. Welcome Calvin!</p>

	<p><strong><strong>Dec 7, 2018</strong></strong> Xuebing won the Scaringe Award from the RNA Society </p>

	<p><strong><strong>Nov 1, 2018</strong></strong> Lab officially starts! </p>

	
	
	<br><br>
</div></div>
				</div>	
			</section>
			<!-- End news Area -->
			
				

			<!-- Start join Area -->
			<section class="offered-area" id="join">
				<div class="container">
					<div style="margin-left:40px;text-align:left">
</div>
<div>
<div>
	<br><br>
	<h2> Join us </h2>
	<br>
	
	<p>We are seeking passionate researchers of all stages (postdocs, graduate students, undergraduates, and research associates) to join our team. Please email me (xw2629@columbia.edu) directly if you are interested in joining our team. 
	</p>
	<p>
	Columbia University undergraduate students: We welcome passionate and creative students to work in the lab. Dr. Xuebing Wu is also a mentor of the Summer Undergraduate Research Fellowship (SURF) program. 
	</p>	
	
	<br><br>
</div></div>
				</div>	
			</section>
			<!-- End join Area -->
			
				


			<!-- start footer Area -->		
			<footer class="footer-area section-gap">
				
					<div class="footer-bottom d-flex justify-content-between align-items-center flex-wrap">
						<!-- Link back to Colorlib can't be removed. Template is licensed under CC BY 3.0. -->
            <p class="footer-text m-0">Copyright &copy;<script>document.write(new Date().getFullYear());</script> All rights reserved | This template is made by <a href="https://colorlib.com" target="_blank">Colorlib</a></p>
            <!-- Link back to Colorlib can't be removed. Template is licensed under CC BY 3.0. -->
						
					</div>
				</div>
			</footer>	
			<!-- End footer Area -->

			<script src="js/vendor/jquery-2.2.4.min.js"></script>
			<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
			<script src="js/vendor/bootstrap.min.js"></script>			
			<script type="text/javascript" src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBhOdIF3Y9382fqJYt5I_sswSrEw5eihAA"></script>
  			<script src="js/easing.min.js"></script>			
			<script src="js/hoverIntent.js"></script>
			<script src="js/superfish.min.js"></script>	
			<script src="js/jquery.ajaxchimp.min.js"></script>
			<script src="js/jquery.magnific-popup.min.js"></script>	
			<script src="js/owl.carousel.min.js"></script>			
			<script src="js/jquery.sticky.js"></script>
			<script src="js/jquery.nice-select.min.js"></script>	
			<script src="js/waypoints.min.js"></script>
			<script src="js/jquery.counterup.min.js"></script>					
			<script src="js/parallax.min.js"></script>	
			<script src="js/mail-script.js"></script>	
			<script src="js/main.js"></script>	
		</body>
	</html>



