	<!DOCTYPE html>
	<html lang="zxx" class="no-js">
	<head>
		<!-- Mobile Specific Meta -->
		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
		<!-- Favicon-->
		<link rel="shortcut icon" href="img/fav.png">
		<!-- Author Meta -->
		<meta name="author" content="codepixer">
		<!-- Meta Description -->
		<meta name="description" content="">
		<!-- Meta Keyword -->
		<meta name="keywords" content="">
		<!-- meta character set -->
		<meta charset="UTF-8">
		<!-- Site Title -->
		<title>Wu Lab @ Columbia | Medicine | Systems Biology </title>

		<link href="https://fonts.googleapis.com/css?family=Poppins:100,200,400,300,500,600,700" rel="stylesheet"> 
			<!--
			CSS
			============================================= -->
			<link rel="stylesheet" href="css/linearicons.css">
			<link rel="stylesheet" href="css/font-awesome.min.css">
			<link rel="stylesheet" href="css/bootstrap.css">
			<link rel="stylesheet" href="css/magnific-popup.css">
			<link rel="stylesheet" href="css/nice-select.css">					
			<link rel="stylesheet" href="css/animate.min.css">
			<link rel="stylesheet" href="css/owl.carousel.css">
			<link rel="stylesheet" href="css/main.css">
		</head>
		<body>

			  <header id="header" id="home">
			    <div class="container">
			    	<div class="row align-items-center justify-content-between d-flex">
				      <div id="logo">
				        <a href="index.html"><img src="img/logo.png" alt="" title="" /></a>
				      </div>
				      <nav id="nav-menu-container">
				        <ul class="nav-menu">
				          <li class="menu-active"><a href="#home">Home</a></li>
				          <li><a href="#research">Research</a></li>
				          <li><a href="#publications">Publications</a></li>
				          <li><a href="#people">People</a></li>
	                                  <li><a href="#funding">Funding</a></li>
				          <li><a href="#Software">Software</a></li>
						  <li><a href="#protocols">Protocols</a></li>
				          <li><a href="#news">News</a></li>
							<li><a href="#join">Join</a></li>

							<!--
				          <li class="menu-has-children"><a href="">Join</a>
				            <ul>
				              <li><a href="generic.html">Generic</a></li>
				              <li><a href="elements.html">Elements</a></li>
				            </ul>
				          </li> -->
				        </ul>
				      </nav><!-- #nav-menu-container -->		    		
			    	</div>
			    </div>
			  </header><!-- #header -->

			<!-- start banner Area -->
			<section class="banner-area relative" id="home">
				<div class="container" >
					<div class="row fullscreen d-flex align-items-center justify-content-end">
						<div class="banner-content col-lg-7 col-md-12">
							<h4 class="text-uppercase"></h4>
							<h2> <font color="#7F97FF">Wu Lab @ Columbia </font>  </h2>
							
							<h2> <font color="#666666"> Decoding and Targeting the Central Dogma </font>  </h2>
							
							<p>
							<font size="4">
								We study the fundamental principles of how genetic information is coded and decoded in health and disease and how to target these processes in cancer, heart disease, aging, and neurodegeneration. 
								To achieve these goals, we utilize cutting edge technologies, including CRISPR, genomics, and machine learning.
								</font>
							</p>
						 <!--	<a href="#research" class="primary-btn header-btn text-uppercase">Learn more</a>  -->
							<br><br><br><br><br><br><br><br><br>
						</div>												
					</div>
				</div>
			</section>
			<!-- End banner Area -->	
			
			
			<!-- Start feature Area --> 
			<section class="feature-area section-gap" id="research">
				<div class="container">
					<h2> RESEARCH </h2> <br> <br>
					<div class="row">
						
						<div class="single-feature col-lg-4">
							<img class="img-fluid" src="img/f1.png" alt="">
						  <h4>Decoding the central dogma</h4>
						  <p> How do cells accurately synthesize a complex proteome from a genome with 98% noncoding DNA? </p>
						  <hr>

							<strong>1</strong>) A unified model for the surveillance of translation in noncoding sequences.<br>
                        See <a href="https://www.nature.com/articles/s41586-023-05946-4">Kesner et al., 2023 <em>Nature</em></a><br>	
							
							<strong>2</strong>) A widespread role of RNA structures in controlling mRNA 3' end processing and stability.<br>
                        See <a href="http://www.cell.com/cell/fulltext/S0092-8674(17)30490-7">Wu & Bartel, 2017 <em>Cell</em></a><br>	
							
				          <strong>3</strong>) How cells suppress pervasive noncoding transcription and how noncoding transcription drives the evolution of new genes.<br>
				          See <a href="http://www.cell.com/abstract/S0092-8674(13)01406-2">Wu & Sharp, 2013 <em>Cell</em></a>, <a href="http://www.nature.com/nature/journal/v499/n7458/full/nature12349.html">Almada & Wu et al. 2013 <em>Nature</em></a> <br>
                          			</div>
						<div class="single-feature col-lg-4">
							<img class="img-fluid" src="img/f2.png" alt="">
							<h4>CRISPR technology & therapy</h4>
							<p>We use CRISPR to control RNAs and proteins for therapeutic applications.<br>
							</p>
							<hr>
					        <strong>1</strong>)  The first genome-wide analysis of Cas9 targeting in mammalian cells.<br>
					        See 
						      <a href="http://www.nature.com/nbt/journal/v32/n7/full/nbt.2889.html">Wu et al. 2014 <em>Nature Biotechnology</em></a><br>
				          <strong>2</strong>) CRISPR/Cas-based technologies for editing the genome, epigenome, and transcriptome.<br>
			            See <a href="http://www.sciencemag.org/content/339/6121/819.short">Cong et al. 2013 <em>Science</em></a>, <a href="http://www.nature.com/nature/journal/v520/n7546/full/nature14299.html">Ran et al. 2015 <em>Nature</em></a>, <a href="http://www.cell.com/cell/fulltext/S0092-8674(16)31153-9">Liu et al. 2016 <em>Cell</em></a>, <a href="http://www.cell.com/cell/fulltext/S0092-8674(18)30049-7">Liu et al. 2018 <em>Cell</em></a>, <a href="https://www.sciencedirect.com/science/article/pii/S0092867418311735">Zhang et al. 2018 <em>Cell</em></a>						</div>
						<div class="single-feature col-lg-4">
							<img class="img-fluid" src="img/f3.png" alt="">
							<h4>Computational biology</h4>
						  <p> We are currently developing interpretable deep learning models for alternative splicing and CRISPR targeting.</p>
							<hr>
							<p><strong>						    1</strong>) A  web server called <a href="http://kplogo.wi.mit.edu/" target="_blank">kpLogo</a> for position-specific short motif analysis.<br>
							  See
							  <a href="http://dx.doi.org/10.1093/nar/gkx323">Wu & Bartel 2017 <em>Nucleic Acids Research</em></a><br>
						      <strong>2</strong>) Network-based global prediction of human disease genes.<br>
						      See
						      <a href="http://msb.embopress.org/content/4/1/189">Wu et al. 2008 <em>Molecular Systems Biology</em></a><br>
				          <strong>3</strong>) Network alignment-based identification of disease families and associated gene modules.<br>
				          See
				          <a href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/25/1/98">Wu et al. 2009 <em>Bioinformatics</em></a></p>
						</div>												
					</div>
				</div>	
			</section>
			<!-- End feature Area -->


			<!-- Start publication Area -->
			<section class="offered-area" id="publications">
				<div class="container">
					<div style="margin-left:40px;text-align:left">
</div>
<div>
<div>
	<br><br>
	<h2> PUBLICATIONS </h2>
	
<div><span style="color:rgb(0,0,255)"><font size="3"><a href="http://scholar.google.com/citations?user=qxEYZ9UAAAAJ"><img src="http://scholar.google.co.uk/intl/en/scholar/feather-72.png" style="display:inline;margin:5px 10px 0px 0px;float:left" width="30" height="60"></a><br>
<a href="http://scholar.google.com/citations?user=qxEYZ9UAAAAJ">Google Scholar Citations<br>
</a></font></span></div>
<span style="font-size:x-large;line-height:34.9090919494629px;background-color:transparent"><span style="font-size:x-large;line-height:34.9090919494629px;background-color:transparent"><span style="font-size:x-large;line-height:34.9090919494629px;background-color:transparent"><br>
</span></span></span></div>
	
	co-first author<sup>*</sup>,
	co-corresponding author<sup>^</sup> <br><br>

		<p><font size="5"><strong><strong>2024</strong></strong></font></p>
	<hr>

	<p>Peiguo Shi, Xuebing Wu<br>
  <a href="https://doi.org/10.1080/15476286.2024.2351657" target="_blank">Programmable RNA targeting with CRISPR-Cas13.</a><br>
  <strong><strong><em>RNA Biology</em></strong></strong>, 2024, 21:1–9<br>
	</p>
	
			<p>Imdadul Haq, Jason C. Ngo, Nainika Roy, Richard L. Pan, Nawsheen Nadiya, Rebecca Chiu, Ya Zhang, Masashi Fujita, 
				Rajesh K. Soni, Xuebing Wu, David A Bennett, Vilas Menon, Marta Olah, Falak Sher<br>
  <a href="https://stemcellres.biomedcentral.com/articles/10.1186/s13287-024-03700-9" target="_blank">An integrated toolkit for human microglia functional genomics.</a><br>
  <strong><strong><em>Stem Cell Research & Therapy</em></strong></strong>, 2024, 15:104<br>
	</p>
	
		<p>Jian Ge, Hongxu Ding, Hongxia Shao, Yuefeng Huang, Xuebing Wu, Jie Sun, Jianwen Que<br>
  <a href="https://www.sciepublish.com/article/pii/134" target="_blank">Single cell analysis of lung lymphatic endothelial cells and lymphatic responses during influenza infection.</a><br>
  <strong><strong><em>Journal of Respiratory Biology and Translational Medicine</em></strong></strong>, 2024, 1:10003<br>
	</p>
	
	<p><font size="5"><strong><strong>2023</strong></strong></font></p>
	<hr>
	
		<p>Yingyu Zhang, Wanwei Zhang, Jingyao Zhao, Takamasa Ito, Jiacheng Jin, Alexis O. Aparicio, Junsong Zhou, Vincent Guichard, Yinshan Fang, Jianwen Que, Joseph F. Urban Jr, Jacob H. Hanna, Sankar Ghosh, Xuebing Wu, Lei Ding, Uttiya Basu & Yuefeng Huang<br>
  <a href="https://www.nature.com/articles/s41590-023-01548-4" target="_blank">m6A RNA modification regulates innate lymphoid cell responses in a lineage-specific manner.</a><br>
  <strong><strong><em>Nature Immunology </em></strong></strong>, 2023, 24:1256–1264 <br>
	</p>
	
	<p> Jordan S Kesner*, Ziheng Chen*, Peiguo Shi, Alexis O Aparicio, Michael R. Murphy, Yang Guo, Aditi Trehan, Jessica E. Lipponen, Yocelyn Recinos, Natura Myeku, Xuebing Wu<br>
  <a href="papers/kesner2023.pdf" target="_blank">Noncoding translation mitigation.</a><br>
  <strong><strong><em>Nature</em></strong></strong>, 2023, 617:395–402 <br>
		See related  <a href="https://www.biorxiv.org/content/10.1101/2022.07.20.500724v1" target="_blank"> preprint </a>
	</p>
	
		<p> Peiguo Shi^, Michael R Murphy, Alexis O Aparicio, Jordan S Kesner, Zhou Fang, Ziheng Chen, Aditi Trehan, Yang Guo, Xuebing Wu^ <br>
  <a href="papers/shi2023.pdf" target="_blank">Collateral activity of the CRISPR/RfxCas13d system in human cells.</a><br>
  <strong><strong><em>Communications Biology</em></strong></strong>, 2023 6:334 <br>
		See related  <a href="https://www.biorxiv.org/content/10.1101/2021.11.30.470032v1" target="_blank"> preprint </a>
	</p>
	
		
	<p><font size="5"><strong><strong>2022</strong></strong></font></p>
	<hr>
	
	<p> Jordan S Kesner*, Ziheng Chen*, Alexis A Aparicio, Xuebing Wu <br>
  <a href="https://www.biorxiv.org/content/10.1101/2022.07.20.500724v1" target="_blank">A unified model for the surveillance of translation in diverse noncoding sequences.</a><br>
  <strong><strong><em>bioRxiv</em></strong></strong>, 2022.07.20.500724; doi: https://doi.org/10.1101/2022.07.20.500724 </p>
	
	
		<p><font size="5"><strong><strong>2021</strong></strong></font></p>
	<hr>
	
	<p> Peiguo Shi^, Michael R Murphy, Alexis O Aparicio, Jordan S Kesner, Zhou Fang, Ziheng Chen, Aditi Trehan, Xuebing Wu^ <br>
  <a href="https://www.biorxiv.org/content/10.1101/2021.11.30.470032v1" target="_blank">RNA-guided cell targeting with CRISPR/RfxCas13d collateral activity in human cells.</a><br>
  <strong><strong><em>bioRxiv</em></strong></strong>, 2021, 2021.11.30.470032; doi: https://doi.org/10.1101/2021.11.30.470032 </p>
	
	
	<p><font size="5"><strong><strong>2018</strong></strong></font></p>
	<hr>

	<p> Cheng Zhang, Silvana Konermann, Nicholas J. Brideau, Peter Lotfy, <u>Xuebing Wu</u>, Scott J. Novick, Timothy Strutzenberg, Patrick R. Griffin, Patrick D. Hsu, Dmitry Lyumkis<br>
  <a href="papers/zhang2018.pdf" target="_blank">Structural basis for the RNA-guided ribonuclease activity of CRISPR-Cas13d.</a><br>
  <strong><strong><em>Cell</em></strong></strong>, 2018, 175:212-223</p>
	
  <p>Josh Tycko, Luis A Barrera, Nicholas Huston, Ari E Friedland, <u>Xuebing Wu</u>, Jonathan S Gootenberg, Omar O Abudayyeh, Vic E Myer, Christopher J Wilson, Patrick D Hsu<br>
  <a href="papers/tycko2018.pdf" target="_blank">Pairwise library screen systematically interrogates Staphylococcus aureus Cas9 specificity in human cells.</a><br>
  <strong><strong><em>Nature Communications</em></strong></strong>, 2018, 9:2962</p>
	
  <p>Wei Jiang*, Yuehua Wei*, Yong Long*, Arthur Owen, Bingying Wang, <u>Xuebing Wu</u>, Shuo Luo, Yongjun Dang, Dengke K. Ma<br>
  <a href="papers/jiang2018.pdf" target="_blank">A genetic program mediates cold-warming response and promotes stress-induced phenoptosis in C. elegans</a><br>
  <strong><strong><em>eLife</em></strong></strong>, 2018, 7:e35037</p>
	
  <p>X Shawn Liu, Hao Wu, Marine Krzisch, <u>Xuebing Wu</u>, John Graef, Julien Muffat, Denes Hnisz, Charles H. Li, Bingbing Yuan, Chuanyun Xu, Yun Li, Dan Vershkov, Angela Cacace, Richard A. Young, 
    Rudolf Jaenisch<br>
  <a href="papers/liu2018.pdf" target="_blank">Rescue of fragile X syndrome neurons by DNA methylation editing of the FMR1 gene</a><br>
  <em><strong><strong>Cell</strong></strong></em>, 2018, 172:979–992<br></p>
	
  <p>Anthony C. Chiu*, Hiroshi I. Suzuki*, <u>Xuebing Wu</u>, Dig B. Mahat, Andrea J. Kriz, Phillip A. Sharp<br>
  <a href="papers/chiu2018.pdf" target="_blank">Transcriptional Pause Sites Delineate Stable Nucleosome-Associated Premature Polyadenylation Suppressed by U1 snRNP</a> <br>
  <strong><strong><em>Molecular Cell</em></strong></strong>, 2018, 69:648–663<br></p>
	
  	<p><font size="5"><strong><strong>2017</strong></strong></font></p>
  	<hr>

	  
<p><u>Xuebing Wu</u>, David P. Bartel<br>
<a href="papers/wu2017cell.pdf" target="_blank">Widespread influence of 3′-end structures on mammalian mRNA processing and stability</a><br>
<strong><strong><em>Cell</em></strong></strong>, 2017, 169: 905–917<br></p>
	
<p><u>Xuebing Wu</u>^, David P. Bartel^ (^co-corresponding author)<br>
  <a href="papers/wu2017nar.pdf" target="_blank"><em>k</em>pLogo: positional <em>k</em>-mer analysis reveals hidden specificity in biological sequences</a><br>
  <strong><strong><em>Nucleic Acids Research</em></strong></strong>, <a href="https://doi.org/10.1093/nar/gkx323" target="_blank">gkx323</a> <br>
  <a href="http://kplogo.wi.mit.edu">http://kplogo.wi.mit.edu</a></p>
	
	<p><font size="5"><strong><strong>2016</strong></strong></font></p>
	<hr>
	
<p>X Shawn Liu*, Hao Wu*, Xiong Ji, Yonatan Stelzer, <u>Xuebing Wu</u>, Szymon Czauderna, Jian Shu, Chikdu S. Shivalila, Daniel Dadon, Richard A. Young, Rudolf Jaenisch<br>
 <a href="papers/liu2016.pdf" target="_blank">Editing DNA methylation in the mammalian genome</a><br>
  <strong><strong><em>Cell</em></strong></strong>, 2016, 167:233-247<br>
	
<p>Xiaochang Zhang, Ming Hui Chen, <u>Xuebing Wu</u>, Andrew Kodani, Jean Fan, Ryan Doan, Manabu Ozawa, Jacqueline Ma, Nobuaki Yoshida, Jeremy F Reiter, Douglas L. Black, Peter V Kharchenko, Phillip A. Sharp, Christopher A. Walsh<br>
<a href="papers/zhang2016.pdf" target="_blank">Cell type-specific alternative splicing governs cell fate in the developing cerebral cortex</a><br>
<strong><strong><em>Cell</em></strong></strong>, 2016, 166:1147-1162</p>
	
	
	<p><font size="5"><strong><strong>2015</strong></strong></font></p>
	<hr>
	
<p>F. Ann Ran*, Le Cong*, Winston X. Yan*, David A. Scott, Jonathan S. Gootenberg, Andrea J. Kriz, Bernd Zetsche, Ophir Shalem, <u>Xuebing Wu</u>, Kira S. Makarova, Eugene V. Koonin, Phillip A. Sharp, Feng Zhang<br>
<a href="papers/ran2015.pdf" target="_blank">In vivo genome editing using Staphylococcus aureus Cas9</a><br>
<em><strong><strong>Nature</strong></strong></em>, 2015, 520:186-191</p>
	
	<p><font size="5"><strong><strong>2014</strong></strong></font></p>
	<hr>
	
<p><u>Xuebing Wu</u>, Andrea J. Kriz, Phillip A. Sharp <br>
  <a href="papers/wu2014qb.pdf" target="_blank">Target specificity of the CRISPR-Cas9 system</a></strong></strong><br><em><strong><strong>Quantitative Biology</strong></strong></em>, 2014, 2:59-70<br>
  </p>
			
<p><u>Xuebing Wu</u>, David A. Scott, Andrea J. Kriz, Anthony C. Chiu, Patrick D. Hsu, Daniel B. 
Dadon, Albert W. Cheng, Alexandro E. Trevino, Silvana Konermann, Sidi Chen, Rudolf Jaenisch, Feng Zhang, Phillip A. Sharp<br>
<a href="papers/wu2014nbt.pdf" target="_blank">Genome-wide binding of the CRISPR endonuclease Cas9 in mammalian cells</a><br>

	<em><strong><strong>Nature Biotechnology</strong></strong></em>, 2014, 32:670-676<br></p>
			
<p>Sidi Chen, Yuan Xue, <u>Xuebing Wu</u>, Le Cong, Arjun Bhutkar, Eric Bell, Feng Zhang, Robert Langer, Phillip A. Sharp<br>
<a href="papers/chen2014gd.pdf" target="_blank">Global microRNA depletion suppresses tumor angiogenesis</a><br>
<strong><strong><em>Genes &amp; Development</strong></strong></em>, 2014, 28:1054-1067 <br>
</p>
			
	<p><font size="5"><strong><strong>2013</strong></strong></font></p>
	<hr>
			
<p><u>Xuebing Wu</u>, Phillip A. Sharp<br>
<a href="papers/wu2013cell.pdf" target="_blank">Divergent transcription: a driving force for new gene origination?</a><br>
<em><strong><strong>Cell</strong></strong></em>, 2013, 155:990-996<br>			
</p>
			
<p>Albert E. Almada*, <u>Xuebing Wu</u>*, Andrea J. Kriz, Christopher B. Burge, Phillip A. Sharp (*equal contribution)<br>
<a href="papers/almada2013.pdf" target="_blank">Promoter directionality is controlled by U1 snRNP and polyadenylation signals</a><br>
<strong><strong><em>Nature</em></strong></strong>, 2013, 499:360–363<br></p>
			
<p> Le Cong, F Ann Ran, David Cox, Shuailiang Lin, Robert Barretto, Naomi Habib, Patrick D Hsu, <u>Xuebing Wu</u>, Wenyan Jiang, Luciano A Marraffini, Feng Zhang<br>
<a href="papers/cong2013.pdf" target="_blank">Multiplex genome engineering using CRISPR/Cas systems</a><strong><strong><em><br>
Science</em></strong></strong>, 2013, 339 (6121):819-823. <br>
</p>
	
<p>Patrick D. Hsu, David A. Scott, Joshua A. Weinstein, F. Ann Ran, Silvana Konermann, Vineeta Agarwala, Yinqing Li, Eli J. Fine, <u>Xuebing Wu</u>, Ophir Shalem, Thomas J. Cradick, Luciano A. Marraffini, Gang Bao, and Feng Zhang<br>
<a href="papers/hsu2013.pdf" target="_blank">DNA Targeting Specificity of the RNA-guided Cas9 Nuclease</a><strong><strong><em><br>
Nature Biotechnology</em></strong></strong>, 2013, 31:827-832
<br></p>
	
<p>Anna Lyubimova, Shalev Itzkovitz, Jan Philipp Junker, Zi Peng Fan,  <u>Xuebing Wu</u>, Alexander van Oudenaarden<br>
<a href="papers/ana2013.pdf" target="_blank">Single-molecule mRNA detection and counting in mammalian tissue</a><strong><strong><em><br>
Nature Protocols</em></strong></strong>, 2013, 8:1743–1758 <a href="https://github.com/xuebingwu/ImageM" target="_blank">(download software)</a></p>
	
<p>Yong Chen, <u>Xuebing Wu</u>, Rui Jiang<br>
<a href="papers/chen2013.pdf" target="_blank">Integrating human omics data to prioritize candidate genes</a><br>
<strong><strong><em>BMC Medical Genomics</em></strong></strong>, 2013, 6:57<br>
</p>
	
	<p><font size="5"><strong><strong>2010</strong></strong></font></p>
	<hr>
	
<p><u>Xuebing Wu</u>, Shao Li<br>
  <a href="papers/wu2010.pdf" target="_blank">Cancer gene prediction using a network approach</a><br>
  Chapter 11 in: <strong><strong><em>Cancer Systems Biology</em></strong></strong> (Ed. Edwin Wang). Series: Chapman &amp; Hall/CRC Mathematical &amp; Computational Biology, USA: CRC Press 2010. [<a href="http://www.amazon.com/Systems-Biology-Chapman-Mathematical-Computational/dp/1439811857" target="_blank"><span style="color:rgb(61,133,198)">Amazon</span></a>]	    
  </pp>
  
<p><font size="5"><strong><strong>2009</strong></strong></font></p>
<hr>

		
        <p> <u>Xuebing Wu</u>, Qifang Liu, Rui Jiang<br>
              <a href="papers/wu2009bioinfo.pdf" target="_blank">Align human interactome with phenome to identify causative genes and networks underlying disease families</a><br>
			<strong><strong><em>Bioinformatics</em></strong></strong>, 2009, 25 (1): 98-104 [<a href="http://bioinfo.au.tsinghua.edu.cn/alignpi/" target="_blank"><span style="color:rgb(61,133,198)">Data</span></a>]<br>
        </p>
			
<p>Wanwan Tang, <u>Xuebing Wu</u>, Rui Jiang, Yanda Li<br>
 <a href="papers/tang2009.pdf" target="_blank">Epistatic module detection for case-control studies: A Bayesian model with a Gibbs sampling strategy</a><br>
 <strong><strong><em>PLoS Genetics</em></strong></strong>, 2009, (5):e1000464<br>
			</p>
			
<p>Rui Jiang, Wanwan Tang, <u>Xuebing Wu</u>, Wenhui Fu<br>
<a href="papers/jiang2009.pdf" target="_blank">A random forest approach to the detection of epistatic interactions in case-control studies</a><br>
<strong><strong><em>BMC Bioinformatics</em></strong></strong>, 2009, 10 (Suppl 1):S65 <br>
			</p>
			
	<p><font size="5"><strong><strong>2008</strong></strong></font></p>
	<hr>
			
<p> <u>Xuebing Wu</u>, Rui Jiang, Michael Q. Zhang, Shao Li<br>
  <a href="papers/wu2008.pdf" target="_blank">Network-based global inference of human disease genes</a><br>
  <strong><strong><em>Molecular Systems Biology</em></strong></strong>, 2008, 4:189.<br> </p>
  <br> 
	<hr>
			<p><font size="5"><strong>Xuebing's undergraduate publications</strong></font></p>
	<hr>
		
			<p><u>Xuebing Wu</u>, Zhirong Sun, Rui Jiang <br>
<a href="papers/wu2008b.pdf" target="_blank">Logic motif of combinatorial control in transcriptional networks</a><br>
<strong><strong><em>Nature Precedings</em></strong></strong>, 2008, https://doi.org/10.1038/npre.2008.2227.1 <br>
			</p>
				
	<p>Zhirong Sun, <u>Xuebing Wu</u><br>
<a href="papers/wu2006.pdf" target="_blank">Systems biology: new interdisciplinary science</a><br>
<strong><strong><em>Communication of China Computer Society</em></strong></strong>, 2006, 2 (3):56-65 (Cover story, review, in Chinese)<br>
			</p>
		
		<p><u>Xuebing Wu</u>, Yuanjie Li, Huaiyu Wang <br>
<a href="papers/wu2005.pdf" target="_blank">An intelligent transportation system based on virtual force</a><br>
<strong><strong><em>Physics and Engineering</em></strong></strong>, 2005, 2 (in Chinese)<br>
			</p>
</div>
</div>
</div>

			</section>
			<!-- End publication Area -->
			
<!-- Start people Area --> 
			<section class="offered-area" id="people">
				<div class="container">
									<h2>Lab members</h2>
<br><br>

					<div class="row">
						<table border="0">
							
	<!--						
			<tr>
				<td> 2019/12/07 </td>
            <td bgcolor="#FFFFFF"><img src="img/2019winter.jpg" height="300" /></td>
				<br><br><br><br>
          </tr>
		-->	
								<tr>
				<td> 2023/09/29 </td>
            <td bgcolor="#FFFFFF"><img src="img/lab-photo-2023.png" height="500" /></td>
				<br><br><br><br>
          </tr>
							
							
			<tr>
            <td width="200" height="200" bgcolor="#FFFFFF"><img src="img/xuebing2024.png" width="180" /></td>
            <td bgcolor="#FFFFFF"><div class="">
		    <br><br>
              <h4 align="left">Xuebing Wu</h4>
              <div align="left">
                <p><br>
                  Assistant Professor <br>
		  Department of Medicine / Cardiology division <br>
			-- Cardiometabolic Genomics Program<br>
		  Department of Systems Biology<br>
		  Herbert Irving Comprehensive Cancer Center <br>
		  Data Science Institute <br>
		 </p>
              </div>
            </div></td>
          </tr>
				
							<!--
													
			<tr>
            <td width="200" height="200" bgcolor="#FFFFFF"><img src="img/alexis_aparicio20190219.jpg" width="180" height="180" /></td>
            <td bgcolor="#FFFFFF"><div class="">
              <h4 align="left">Alexis Aparicio</h4>
              <div align="left">
		      <br> 
                <p>Technician<br>
		   BA in Biochemistry and Molecular Biology, Boston University
		      </p>
		      
							<p>
							</p>
              </div>
            </div></td>
          </tr>
			-->
							
																	
			<tr>
            <td width="200" height="200" bgcolor="#FFFFFF"><img src="img/michaelmurphy20190301.jpg" width="180" height="180" /></td>
            <td bgcolor="#FFFFFF"><div class="">
              <h4 align="left">Michael Murphy</h4>
              <div align="left">
		      <br> 
                <p>Associate Research Scientist (Senior Postdoc) <br>
			Ph.D. in Molecular, Cellular, and Developmental Biology, City University of New York
		      </p>
		      
							<p>
							</p>
              </div>
            </div></td>
          </tr>
							
													<tr>
            <td width="200" height="200" bgcolor="#FFFFFF"><img src="img/PeiguoShi20191101.jpg" width="180" height="180" /></td>
            <td bgcolor="#FFFFFF"><div class="">
              <h4 align="left">Peiguo Shi</h4>
              <div align="left">
		      <br> 
                <p>Associate Research Scientist<br>
			Ph.D. in Cell Biology, University of the Chinese Academy of Sciences, Beijing, China
		      </p>
		      
							<p>
							</p>
              </div>
            </div></td>
          </tr>
			<!--				
<tr>
	<td width="200" height="200" bgcolor="#FFFFFF"><img src="img/KatherineLiu20191101.jpg" width="180" height="180" /></td>
	<td bgcolor="#FFFFFF"><div class="">
	<h4 align="left">Katherine Liu</h4>
	<div align="left">
	<br> 
	<p>Undergraduate student (Biomedical Engineering)<br>
		Columbia University
	</p>
	<p> liu.katherine@columbia.edu</p>
	<p>
	</p>
	</div>
	</div></td>
</tr>
				
<tr>
	<td width="200" height="200" bgcolor="#FFFFFF"><img src="img/yixuanli20191101.jpg" width="180" height="180" /></td>
	<td bgcolor="#FFFFFF"><div class="">
	<h4 align="left">Yixuan Li</h4>
	<div align="left">
	<br> 
	<p>Undergraduate student (Computer Science)<br>
		Columbia University
	</p>
	<p> yl3803@columbia.edu </p>
	<p>
	</p>
	</div>
	</div></td>
</tr>
  

							
<tr>
	<td width="200" height="200" bgcolor="#FFFFFF"><img src="img/ZanisFang20191101.jpg" width="180" height="180" /></td>
	<td bgcolor="#FFFFFF"><div class="">
	<h4 align="left">Zanis Fang</h4>
	<div align="left">
	<br> 
	<p>Postdoctoral Research Scientist<br>
	   PhD in Genetics, Fudan University <br>
	   MS in Biostatistics, Data Science Institute Scholar, 2019, Columbia University
	</p>
	<p>
	</p>
	</div>
	</div></td>
</tr>	

							<tr>
	<td width="200" height="200" bgcolor="#FFFFFF"><img src="img/shruti20200211.jpg" width="180" height="180" /></td>
	<td bgcolor="#FFFFFF"><div class="">
	<h4 align="left">Shruti Verma</h4>
	<div align="left">
	<br> 
	<p>Egleston Scholar, Class of 2023, Columbia Engineering <br> Data Science Institute Scholar, 2020
	   <br>
		</p>
	<p>
	</p>
	</div>
	</div></td>
</tr>	
	
	
							<tr>
	<td width="200" height="200" bgcolor="#FFFFFF"><img src="img/jordankesner20200801.jpg" width="180" height="180" /></td>
	<td bgcolor="#FFFFFF"><div class="">
	<h4 align="left">Jordan Kesner</h4>
	<div align="left">
	<br> 
	<p>
	   Postdoc Research Scientist <br>
	   PhD in Systems Biology, Columbia University <br>
	   MS in Biotechnology, Columbia University <br>
	   BS in Molecular Biology, Cornell University
       	</p>
	<p>
	</p>
	</div>
	</div></td>
</tr>	
-->
							<tr>
	<td width="200" height="220" bgcolor="#FFFFFF"><img src="img/AditiTrehan.jpg" width="180" height="220" /></td>
	<td bgcolor="#FFFFFF"><div class="">
	<h4 align="left">Aditi Trehan</h4>
	<div align="left">
	<br> 
	<p>PhD student, Integrated Program in Cellular, Molecular, and Biomedical Studies, Columbia University<br>
	   MS in Biotechnology, Johns Hopkins University <br>
	   BS in Neurobiology & Physiology and Cell, Molecular & Developmental Systems Biology, Purdue University
       	</p>
	<p>
	</p>
	</div>
	</div></td>
</tr>	
							
<!--
		
<tr>
	<td width="200" height="220" bgcolor="#FFFFFF"><img src="img/JonathanAlgoo.jpg" width="180" height="180" /></td>
	<td bgcolor="#FFFFFF"><div class="">
	<h4 align="left">Jonathan Algoo</h4>
	<div align="left">
	<br> 
	<p>PhD student (rotation), Integrated Program in Cellular, Molecular, and Biomedical Studies, Columbia University<br>
	BA in Chemistry and Communications of Science and Technology, Vanderbilt University <br>
       	</p>
	<p>
	</p>
	</div>
	</div></td>
</tr>	

		
<tr>
	<td width="200" height="248" bgcolor="#FFFFFF"><img src="img/Yueyin11142021.jpg" width="180" height="223" /></td>
	<td bgcolor="#FFFFFF"><div class="">
	<h4 align="left">Yue Yin</h4>
	<div align="left">
	<br> 
	<p>Bachelor of Arts, Physics, Class of 2024,  <br> 
		Barnard College, Columbia University<br>
       	</p>
	<p>
	</p>
	</div>
	</div></td>
</tr>	
-->

							
							<tr>
	<td width="200" height="200" bgcolor="#FFFFFF"><img src="img/YangGuo-20220418.jpg" width="180" height="180" /></td>
	<td bgcolor="#FFFFFF"><div class="">
	<h4 align="left">Yang Guo</h4>
	<div align="left">
	<br> 
           <p>Postdoc Research Scientist<br>
			PhD in Biochemistry, Zhejiang University, China
		      </p>
	<p>
	</p>
	</div>
	</div></td>
</tr>	


									<tr>
	<td width="200" height="200" bgcolor="#FFFFFF"><img src="img/colin2023.JPG" width="180" height="180" /></td>
	<td bgcolor="#FFFFFF"><div class="">
	<h4 align="left">Colin Kalicki</h4>
	<div align="left">
	<br> 
	<p>
	   PhD student, Systems Biology<br>
	   BS in Bioengineering, Stanford University
       	</p>
	<p>
	</p>
	</div>
	</div></td>
</tr>	



<tr>
	<td width="200" height="200" bgcolor="#FFFFFF"><img src="img/feiyue2024.png" width="180" height="180" /></td>
	<td bgcolor="#FFFFFF"><div class="">
	<h4 align="left">Feiyue Yang</h4>
	<div align="left">
	<br> 
	<p>
	   PhD student, Systems Biology<br>
	   BS in Biological Science (major) and Data Science and Big Data Technology (minor), China Agricultural University
       	</p>
	<p>
	</p>
	</div>
	</div></td>
</tr>	


<!--

<tr>
	<td width="200" height="200" bgcolor="#FFFFFF"><img src="img/ho2024.jpeg" width="180" height="180" /></td>
	<td bgcolor="#FFFFFF"><div class="">
	<h4 align="left">Sebastian Ho</h4>
	<div align="left">
	<br> 
	<p>
	   Rotation student, Nutritional and Metabolic Biology<br>
	   BS in Biochemistry, University of Pittsburgh 
       	</p>
	<p>
	</p>
	</div>
	</div></td>
</tr>	

-->
	
<tr>
	<td width="200" height="200" bgcolor="#FFFFFF"><img src="img/SikandarAzam2024.jpg" width="180" height="180" /></td>
	<td bgcolor="#FFFFFF"><div class="">
	<h4 align="left">Sikandar Azam</h4>
	<div align="left">
	<br> 
	<p>
	   Associate Research Scientist (Senior Postdoc)<br>
	   PhD in Biochemistry and Molecular Biology, Dalian Medical University
       	</p>
	<p>
	</p>
	</div>
	</div></td>
</tr>	
		
						</table>
						
						
					</div>
				</div>	
			</section>
			<!-- End feature Area -->
			
			<!-- Start lab alumni Area -->
			<section class="offered-area" id="alumni">
				<div class="container">
					<div style="margin-left:40px;text-align:left">
</div>
<div>
<div>
	<br><br>
	<h2> Lab Alumni </h2>
	<br>
	<p><strong><strong>Jordan Kesner</strong></strong> 8/1/2020-10/1/2023, PhD student -> Postdoc </p> 

	<p><strong><strong>Katherine Da</strong></strong> 9/1/2023-11/30/2023, Visiting undergraduate student, Dartmouth College </p> 

	<p><strong><strong>Yue Yin</strong></strong> 11/01/2021-07/01/2022, Barnard Summer Research Institute, undergraduate student</p>

	<p><strong><strong>Alexis Aparicio</strong></strong> 02/19/19-08/19/2022, Technician B</p>
	
        <p><strong><strong>Jonathan Algoo</strong></strong> 9/20/21-12/14/2021, Rotation student, integrated program</p>

	<p><strong><strong>Zhou "Zanis" Fang </strong></strong> 11/01/19-12/06/2021, DSI Scholar -> Postdoc </p>

	<p><strong><strong>Ziheng "Calvin" Chen </strong></strong> 1/15/19-8/10/2020, Master's student, Columbia BME</p>

	<p><strong><strong>Qijin Yin </strong></strong> 11/25/19-5/10/2020, Visiting PhD student from Tsinghua University</p>

	<p><strong><strong>Ellen Alt </strong></strong> 5/28/19-8/1/19, Barnard Summer Research Institute, undergraduate student</p>
	<br><br>
</div></div>
				</div>	
			</section>
			<!-- End news Area -->

<!-- Start funding Area --> 
	<section class="offered-area" id="funding">
		<div class="container">
					<h2>Funding</h2>
					<br><br>
			<div class="row">
		
          			  <td bgcolor="#FFFFFF"><img src="img/funding.png" height="300" /></td>
				
			</div>
		</div>	
	</section>
			
			<!-- Start Software Area -->
			<section class="offered-area" id="Software">
				<div class="container">
					<div style="margin-left:40px;text-align:left">
</div>
<div>
<div>
	<br><br>
	<h2> Software </h2>
	
<div>
<div>
<div>
	<div><br><br><font color="#e69138"><font size="4"><strong><strong><a href="https://github.com/xuebingwu" target="new">Lab GitHub</a></strong></strong></font></font><font size="4" color="#e69138"><strong><strong><br><br><br>
</strong></strong></font></div>
	
<div><font color="#e69138"><font size="4"><strong><strong><em>k</em>pLogo</strong></strong></font></font><font size="4" color="#e69138"><strong><strong><br>
</strong></strong></font></div>
<div>
<hr>
</div>
<font size="4" color="#e69138"><span style="line-height:22px"><strong><strong>
<table width="100%">
<tbody>
<tr>
<td><br>
</td>
<td width="80%">
<div><strong><strong>
<table style="display:inline!important" width="100%">
<tbody style="display:inline!important">
<tr style="display:inline!important">
<td style="display:inline!important" width="80%">
<div style="display:inline!important"><font size="2" color="#000000"><span style="font-weight:normal">
A webserver for positional k-mer analysis<strong><strong><br>
</strong></strong><strong><strong>Links:</strong></strong> <a href="http://kplogo.wi.mit.edu" target="_blank">http://kplogo.wi.mit.edu</a>
			
			<br>
			<strong><strong>Citation: </strong></strong></span></font><font size="2" color="#000000"><span style="font-weight:normal"><strong><strong><font size="4" color="#e69138"><span style="line-height:22px"><strong><strong><strong><strong><font size="2" color="#000000"><span style="font-weight:normal">Wu &amp;</span></font></strong></strong></strong></strong></span></font></strong></strong><font size="4" color="#e69138"><span style="line-height:22px"><font size="2" color="#000000"><span style="font-weight:normal"> Bartel (2017)</span></font></span></font><strong><strong><font size="4" color="#e69138"><span style="line-height:22px"><strong><strong><strong><strong><font size="2" color="#000000"><span style="font-weight:normal"> <em><strong><strong>Nucleic Acids Res</strong></strong></em>, <strong><strong>gkx323<br>
</strong></strong></span></font></strong></strong></strong></strong></span></font></strong></strong></span></font></div>
</td>
</tr>
</tbody>
</table>
</strong></strong></div>
</td>
</tr>
</tbody>
</table>
</strong></strong></span></font>
	
<table style="display:inline!important" width="100%">
<tbody style="display:inline!important">
<tr style="display:inline!important">
<td style="display:inline!important" width="80%"><br>
<br>
</td>
</tr>
</tbody>
</table>
</strong></strong></div>
</td>
</tr>
</tbody>
</table>
</strong></strong></div>
</td>
</tr>
</tbody>
</table>
</strong></strong></span></font><font size="4" color="#e69138"><strong><strong>CIPHER</strong></strong></font>
<div>
<hr>
</div>
<div><font size="4" color="#e69138"><span style="line-height:22px"><strong><strong>
<table width="100%">
<tbody>
<tr>
<td><img alt="" src="img/cipher.jpg" style="border:10px solid white" height="158"></td>
<td width="80%">
<div><strong><strong>
<table style="display:inline!important" width="100%">
<tbody style="display:inline!important">
<tr style="display:inline!important">
<td style="display:inline!important" width="80%">
<div style="display:inline!important"><font size="2" color="#000000"><span style="font-weight:normal"><strong><strong>CIPHER</strong></strong>, which stands for Correlating interactome and phenome 
networks to predict disease genes, is a computational framework we 
proposed to prioritize human disease genes. It was one of the first 
studies to explore interactome-phenome wide gene-disease relationships, 
and generated the first comprehensive genetic landscape of human 
disease, connecting 5080 human disease phenotypes with 14433 human genes.
			<br>
<br>
			<strong><strong>Links:</strong></strong> Click here to <a href="http://bioinfo.au.tsinghua.edu.cn/jianglab/cipher/" target="_blank"> Download dataset </a>
			or <a href="http://bioinfo.au.tsinghua.edu.cn/cipher/cipher_search.html" target="_blank"> Explore predictions </a> (try <a href="http://rulai.cshl.edu/tools/cipher/" target="_blank"> mirror at CSHL </a>)
			
			<br>
<br>
			<strong><strong>Citation:</strong></strong> Wu X, Jiang R, Zhang MQ, Li S (2008) Network-based global inference of human disease genes. Molecular Systems Biology, 4:189</span></font><strong><strong>
<table style="display:inline!important" width="100%">
<tbody style="display:inline!important">
<tr style="display:inline!important">
<td style="display:inline!important" width="80%"><br>
</td>
</tr>
</tbody>
</table>
</strong></strong></div>
</td>
</tr>
</tbody>
</table>
</strong></strong></div>
</td>
</tr>
</tbody>
</table>
</strong></strong></span></font></div>
<br>









<div><font size="4" color="#e69138"><strong><strong><br>
</strong></strong></font></div>
<span style="color:rgb(255,153,0)"><font size="4">
AlignPI
</font></span>
<div>
<hr>
<br>
</div>
<div><font size="4" color="#e69138"><span style="line-height:22px"><strong><strong>
<table width="100%">
<tbody>
<tr>
<td><img alt="" src="img/alignpi.gif" style="border:10px solid white" height="158">
</td>
<td width="80%">
<br>
<div><font style="font-weight:normal" size="2" color="#000000"><strong><strong>AlignPI</strong></strong> is the abbreviation of "Align Phenome &amp; Interactome." We have performed the first alignment of human phenome and interactome network, and identified 39 bi-modules and made predictions of candidate genes for 70 diseases. Bi-module consists of two inter-connected modules: a network of diseases and a network of genes.&nbsp; <br>
<br>
<strong><strong>Links</strong></strong>: The </font><font style="font-weight:normal" size="2" color="#000000"><font size="4" color="#e69138"><span style="line-height:22px"><strong><strong><strong><strong><font size="2" color="#000000"><span style="font-weight:normal"><a href="http://bioinfo.au.tsinghua.edu.cn/alignpi/" target="_blank">AlignPI</a></span></font></strong></strong></strong></strong></span></font> website provides a search interface for all identified bi-modules and candidate gene predictions, and corresponding functional enrichment analysis.&nbsp; <br>
<br>
<strong><strong>Citation</strong></strong>: Wu X, Liu Q, Jiang R (2009) Align human interactome with phenome to identify causative genes and networks underlying disease families. Bioinformatics, 25 (1): 98-104</font></div>
</td>
</tr>
</tbody>
</table>
</strong></strong></span></font></div>
<br>
<div><font size="4" color="#e69138"><span style="line-height:22px"><strong><strong>
<div><font size="4" color="#e69138"><span style="line-height:22px"><strong><strong>
<table width="100%">
<tbody>
<tr>
<td><br>
</td>
<td width="80%"><br>
</td>
</tr>
</tbody>
</table>
</strong></strong></span></font></div>
</strong></strong></span></font></div>
</div>
</div>
</div>

				</div>	
			</section>
			<!-- End Software Area -->
			
		
			<!-- Start protocols Area -->
			<section class="offered-area" id="protocols">
				<div class="container">
					<div style="margin-left:40px;text-align:left">
</div>
<div>
<div>
	<br><br>
	<h2> Protocols </h2>
	<br>
	
	<p><a href="protocols/2P-seq-XW.pdf" target="_blank">4sU-2P-Seq and DIM-2P-seq protocol (Mar 2017) (PDF)  </a></p>
	
	<br><br>
</div></div>
				</div>	
			</section>
			<!-- End protocols Area -->

			<!-- Start news Area -->
			<section class="offered-area" id="news">
				<div class="container">
					<div style="margin-left:40px;text-align:left">
</div>

					
<div>
<div>
	<br><br>
	<h2> Lab news </h2>

	<!-- -->

	<p><strong><strong>6/24/24</strong></strong> Xuebing speaks at the 50th Sharp lab reunion symposium “Half A Century of Sharp Minds”</p>

	<p><strong><strong>6/7/24</strong></strong> Xuebing speaks at the 2024 <a href="https://commonfund.nih.gov/highrisk/symposium" target="_blank">High-Risk, High-Reward Research (HRHR) Symposium</a> at Bethesda</p>

	<p><strong><strong>5/19/24</strong></strong> Peiguo's <a href="https://doi.org/10.1080/15476286.2024.2351657" target="_blank">Cas13 review</a> was published online </p>

	<p><strong><strong>5/13/24</strong></strong> Xuebing was selected as a 2024 <a href="https://www.cuimc.columbia.edu/news/schaefer-scholars" target="_blank">Schaefer Research Scholar</a></p>

	<p><strong><strong>4/19/24</strong></strong> Xuebing speaks at the first <a href="https://www.publichealth.columbia.edu/news/healthy-aging-summit-unites-medical-center-shared-goals" target="_blank"> Healthspan Extension Summit at Columbia </a> </p>

	<p><strong><strong>4/3/24</strong></strong> Xuebing is invited to speak at the <a href="https://rnainnovation.med.upenn.edu/event/rnadaypenn/" target="_blank">inaugural RNADay@Penn 2024 </a></p>

	<p><strong><strong>3/26/24</strong></strong> We received the <a href="https://pershingsquarefoundation.org/2024/03/26/press-release-the-pershing-square-foundation-mind-prize-awarded-to-seven-creative-researchers-paving-the-way-to-a-deeper-and-transformative-understanding-of-the-brain/" target="_blank">MIND Prize</a> to test a new idea for treating neurodegeneration</p>
	
	<p><strong><strong>2/1/24</strong></strong> We received our first R01 grant from NHLBI! </p>

	<p><strong><strong>1/8/24</strong></strong> We welcomed rotation students Feiyue Yang (Systems Biology) and Sebastian Ho (Nutritional & Metabolic Bio)</p>

	<p><strong><strong>12/20/23</strong></strong> We received a grant from the <a href="https://www.stxbp1disorders.org/news/orphan-disease-center-2023-mdbr" target="_blank">Million Dollar Bike Ride (MDBR)</a> Grant Program to dissect STXBP1 3’ UTR regulation for therapeutic development </p>
			
	<p><strong><strong>12/19/23</strong></strong> We received a grant from the <a href="https://www.mandlresearch.org/recent-news-1" target="_blank">Ines Mandl Research Foundation (IMRF)</a> to develop an off-the-shelf gene editing therapy for Marfan Syndrome</p>

	<p><strong><strong>11/17/23</strong></strong> We received a grant from the <a href="https://curealz.org/research/translational/studies-of-alternative-neurodegenerative-pathways/noncoding-translation-feedback-loop-in-alzheimers-disease" target="_blank"> Cure Alzheimer’s Fund </a> to study noncoding translation and feedback loops in Alzheimer's disease!</p>

	<p><strong><strong>11/10/23</strong></strong> Xuebing is invited to give a seminar at Duke Biology and RNA Center @ Durham</p>

	<p><strong><strong>11/9/23</strong></strong> Xuebing is invited to give a seminar at NIEHS @ Durham</p>

	<p><strong><strong>10/17/23</strong></strong> Xuebing is invited to give a seminar at Koch Institute/MIT</p>

	<p><strong><strong>10/14/23</strong></strong> Xuebing gave a talk at EMBL@Heidelberg, Germany, during the 2023 non-coding genome symposium</p>

	<p><strong><strong>10/13/23</strong></strong> Xuebing is invited to give a seminar at Heidelberg University Hospital @ Heidelberg, Germany</p>

	<p><strong><strong>10/11/23</strong></strong> Xuebing is invited to give a seminar at Max Planck Institute for Molecular Genetics @ Berlin, Germany</p>

	<p><strong><strong>9/20/23</strong></strong> We received the inaugural <a href="https://www.afar.org/news/announcing-afars-2023-glenn-foundation-discovery-award-recipients" target="_blank"> Glenn Foundation Discovery Award </a> !</p>

	<p><strong><strong>9/18/23</strong></strong> Our NCI admin supplement grant with Que lab has been funded!</p>

	<p><strong><strong>9/13/23</strong></strong> We welcome visiting student Katherine Da, Biology and CS @ Dartmouth College</p>

	<p><strong><strong>9/4/23</strong></strong> We receive a 2nd grant from <a href="https://twitter.com/impetusgrants/status/1698749742006587612" target="_blank"> Longevity Impetus Grants </a> !</p>

	<p><strong><strong>8/28/23</strong></strong> We welcome graduate student Colin Kalicki for his 1st rotation</p>

	<p><strong><strong>7/31/23</strong></strong> Xuebing gave a talk in the Basic Cardiovascular Sciences (BCVS) conference in Boston </p>

	<p><strong><strong>7/3/23</strong></strong> Our collaborative work with Yuefeng Huang lab was published at <a href="https://www.nature.com/articles/s41590-023-01548-4" target="_blank"> Nature Immunology </a></p>

	<p><strong><strong>7/1/23</strong></strong> We receive a grant from HICCC/Inter/Intra-Programmatic Pilot Program (IPPP) to target lung cancer using CRISPR/Cas13 </p>

 	<p><strong><strong>5/30-6/4/23</strong></strong> Xuebing, Michael, and Jordan attended 2023 annual RNA meeting at Singapore. Michael won a poster award. Xuebing gave a talk on RPL3L ribosomes. </p>

	<p><strong><strong>4/19/23</strong></strong> Xuebing was invited to visit U of Chicago by postdocs at the Department of Human Genetics </p>
	
	<p><strong><strong>4/12/23</strong></strong> Our paper titled <a href="https://www.nature.com/articles/s41586-023-05946-4" target="_blank">noncoding translation mitigation</a> has been published online at Nature. See CUIMC news release <a href="https://www.cuimc.columbia.edu/news/illuminating-tumor-cells-dark-proteins" target="_blank"> Illuminating tumor cells with dark proteins </a>
 </p>

	<p><strong><strong>3/28/23</strong></strong> Our paper on <a href="https://www.nature.com/articles/s42003-023-04708-2" target="_blank">Cas13 collateral activity</a> has been published online at Communications Biology </p>
	
	<p><strong><strong>3/21/23</strong></strong> Jordan has been selected as one of the two recipients of the 2023 Dean’s Award for Excellence in Research! Congratulations, Jordan! </p>
	
	<p><strong><strong>10/14/22</strong></strong> Xuebing and Michael each gave an invited virtual talk to the Toronto RNA Club </p>

	<p><strong><strong>9/29/22</strong></strong> Xuebing gave an invited virtual seminar at the University of Texas at Dallas </p>

	<p><strong><strong>9/6-9/22</strong></strong> Xuebing and Michael each gave a talk at the 18th translational control conference at CSHL </p>
	
	<p><strong><strong>8/31/22</strong></strong> Xuebing gave an invited talk at the first <a href="https://www.faseb.org/meetings-and-events/src-events/the-rna-associated-mechanisms-conference-in-immunity-and-disease" targt="_blank">FASEB conference on RNA and immunity</a>. </p>

	<p><strong><strong>8/25/22</strong></strong> Congrats Dr. Jordan Kesner! First PhD from the lab! </p>

	<p><strong><strong>7/21/22</strong></strong> Lab releases the 2nd <a href="https://www.biorxiv.org/content/10.1101/2022.07.20.500724v1" target="_blank">bioRxiv preprint</a>, led by Jordan, Ziheng, and Alex </p>

	<p><strong><strong>6/4/22</strong></strong> Jordan gave a talk at RNA society meeting 2022 </p>
	
	<p><strong><strong>5/25/22</strong></strong> Congrats to Aditi for passing her qualifying exam! </p>

	<p><strong><strong>5/4/22</strong></strong> Xuebing gave an invited talk at Weill Cornell and MSKCC </p>
	
	<p><strong><strong>5/3/22</strong></strong> Xuebing gave an invited virtual talk at Prevail Therapeutics / Eli Lilly and Company </p>

	<p><strong><strong>4/18/22</strong></strong> Lab welcomes Yang Guo as a postdoctoral research scientist !</p>

	<p><strong><strong>11/30/21</strong></strong> First preprint from the lab, led by Peiguo: <a href="https://www.biorxiv.org/content/10.1101/2021.11.30.470032v1" target="_blank">RNA-guided cell targeting with CRISPR/RfxCas13d collateral activity in human cells
 </a></p>

	<p><strong><strong>11/16/21</strong></strong> Lab welcomes Barnard/Columbia undergraduate student Yue Yin!</p>

	<p><strong><strong>10/14/21</strong></strong> Lab receives <a href="https://impetusgrants.org/" target="_blank">Longevity Impetus Grants </a> to study aging!</p>

	<p><strong><strong>9/20/21</strong></strong> Lab welcomes Jonathan Algoo as a rotation student in the Fall!</p>

        <p><strong><strong>6/21/21</strong></strong> Lab receives <a href="https://www.cuimc.columbia.edu/news/three-vp-s-scientists-selected-paul-marks-scholars" target="_blank"> 2021 Paul A. Marks Scholars Award </a>, which provides unrestricted fund for 3 years </p>

	<p><strong><strong>6/3/21</strong></strong> Lab receives <a href="https://www.cancer.columbia.edu/news/new-grant-awardees-push-innovation-precision-medicine" target="_blank"> Columbia Precision Medicine Pilot Grant </a>, which supports our study on specialized ribosomes in the heart</p>

	<p><strong><strong>5/27/21</strong></strong> Aditi Trehan became PhD student #2 of the Wu lab! </p>

	<p><strong><strong>5/18/21</strong></strong> Lab receives <a href="https://psscra.org/prize/2021-winners/" target="_blank"> the 2021 Pershing Square Sohn Prize for Young Investigators in Cancer Research</a>, which supports our study on cancer-targeting using CRISPR/Cas13 and machine learning. Also see <a href="https://www.cancer.columbia.edu/news/cancer-innovation-center-pershing-square-sohn-prizewinners" target="_blank">CUIMC News</a></p>

	<p><strong><strong>4/12/21</strong></strong> Aditi Trehan started her rotation in the lab! </p>

	<p><strong><strong>10/6/20</strong></strong> Lab receives <a href="https://commonfund.nih.gov/newinnovator/AwardRecipients" target="_blank"> NIH Director's New Innovator Award (DP2) </a> for genome-scale study of mRNA noncoding functions using CRISPR/Cas13. Also see <a href="https://www.cuimc.columbia.edu/news/cuimc-researchers-receives-nih-high-risk-high-reward-grant/#wu" target="_blank">CUIMC News</a></p>

	<p><strong><strong>9/1/20</strong></strong> Zanis joined the lab as a Postdoc! </p>

	<p><strong><strong>8/3/20</strong></strong> The Wu lab welcomes our first PhD student Jordan Kesner! </p>

	<p><strong><strong>6/15/20</strong></strong> Xuebing is named a <a href="https://www.pewtrusts.org/en/about/news-room/press-releases-and-statements/2020/06/15/5-pew-stewart-scholars-selected-to-spearhead-innovative-cancer-research" target="_blank">2020 Pew-Stewart Scholar for Cancer Research</a>, which will support our study of how cancer cells reprogram mRNA structures on a genome-scale. Also see <a href="https://cancer.columbia.edu/newly-named-pew-stewart-scholar-dr-xuebing-wu-focuses-therapeutic-targets-breast-cancer" target="_blank">Columbia HICCC News</a> </p>

        <p><strong><strong>4/1/20</strong></strong> Lab's COVID-19 research highlighted in HICCC news article <a href="https://cancer.columbia.edu/columbia-university-cancer-researchers-fight-against-covid-19" target="_blank">Columbia University Cancer Researchers in the Fight Against COVID-19</a></p>

	<p><strong><strong>2/21/20</strong></strong> Xuebing's faculty profile/interview article at <a href="https://systemsbiology.columbia.edu/news/decoding-and-targeting-rna-with-crispr" target="_blank">Dept. Systems Biology</a> and <a href="https://cancer.columbia.edu/xuebing-wu-phd-decoding-and-targeting-rna-crispr" target="_blank">Cancer Center / HICCC</a></p>

        <p><strong><strong>2/10/20</strong></strong> Shruti Verma joined the lab from Columbia Engineering</p>

        <p><strong><strong>12/17/19</strong></strong> Xuebing is named <a href="https://systemsbiology.columbia.edu/news/highly-cited-researchers" target="_blank">Highly Cited Researcher</a> in 2019 by Web of Science</p>

        <p><strong><strong>12/03/19</strong></strong> Michael's <a href="https://professional.heart.org/en/research-programs/application-information/postdoctoral-fellowship" target="_blank">AHA postdoc fellowship</a> application was funded. Congratulations!</p>
	
        <p><strong><strong>11/25/19</strong></strong> Qijin Yin started as a visiting scholar. Welcome Qijin! </p>

	<p><strong><strong>11/01/19</strong></strong> Welcome four new members to the lab: Peiguo Shi (postdoc), Zanis Fang (MS/DSI Scholar), Katherine Liu and Yixuan Li (undergrad) </p>

	<p><strong><strong>10/24/19</strong></strong> Lab moves to our permanent space in P&S</p>

	<p><strong><strong>06/28/19</strong></strong> Xuebing gave a talk at the RNA therapeutics conference at UMass Med School.</p>
	
	<p><strong><strong>06/14/19</strong></strong> Xuebing gave a video-taped talk at the RNA Society Annual Meeting in Krakow, Poland.</p>
	
	<p><strong><strong>05/28/19</strong></strong> Ellen Alt joined the lab as a summer research student. Welcome Ellen!</p>

	<p><strong><strong>03/01/19</strong></strong> Michael Murphy joined the lab as a staff associate. Welcome Michael!</p>

	<p><strong><strong>02/19/19</strong></strong> Alexis Aparicio joined the lab as our first Technician. Welcome Alex!</p>

	<p><strong><strong>01/15/19</strong></strong> Ziheng (Calvin) Chen, a graduate student in the BME Master of Science program joined the lab. Welcome Calvin!</p>

	<p><strong><strong>12/07/18</strong></strong> Xuebing won the <a href="https://www.rnasociety.org/the-rna-society-scaringe-young-scientist-award" target="_blank"> The RNA Society/Scaringe Young Scientist Award 2019 </a> </p>

	<p><strong><strong>11/01/18</strong></strong> Lab officially started! </p>

	
	
	<br><br>
</div></div>
				</div>	
			</section>
			<!-- End news Area -->
			
				

			<!-- Start join Area -->
			<section class="offered-area" id="join">
				<div class="container">
					<div style="margin-left:40px;text-align:left">
</div>
<div>
<div>
	<br><br>
	<h2> Join us </h2>
	<br>
	
	<p>We are seeking passionate researchers of all stages (postdocs, graduate students, undergraduates, and research associates) to join our team. 
	</p>
	<p>
	<strong>Graduate student looking for rotation or postdoc opportunity</strong>: please email me (xuebing.wu@columbia.edu) directly. 
		</p>
	<p>
	<strong>Graduate programs we are affiliated with</strong>: <a href="https://www.gsas.cuimc.columbia.edu/integrated-program-cellular-molecular-and-biomedical-studies" target="_blank">Integrated Program in Cellular, Molecular, and Biomedical Studies (CMBS)</a>; <a href="https://systemsbiology.columbia.edu/how-to-apply" target="_blank">Systems Biology</a> (also a track in the CMBS program); <a href="https://www.gsas.cuimc.columbia.edu/nutritional-and-metabolic-biology" target="_blank">Nutritional and Metabolic Biology</a>. 
		</p>
	<strong>Undergraduate students</strong>: If you are interested in becoming a graduate student in the lab,
		you need to first get admitted to one of the <a href="https://www.gsas.cuimc.columbia.edu/degree-programs" target="_blank">PhD programs at Columbia</a>.
		Once you are admitted, you can choose to rotate or join the lab. We also welcome passionate and creative undergraduate students who can work in the lab full-time during summer or as a visiting student during regular semester. 
		Dr. Xuebing Wu is a mentor of the Summer Undergraduate Research Fellowship (SURF) program and the Barnard Summer Research Institute. 
	</p>	
	
	<br><br>
</div></div>
				</div>	
			</section>
			<!-- End join Area -->
			
				


			<!-- start footer Area -->		
			<footer class="footer-area section-gap">
				
					<div class="footer-bottom d-flex justify-content-between align-items-center flex-wrap">
						<!-- Link back to Colorlib can't be removed. Template is licensed under CC BY 3.0. -->
            <p class="footer-text m-0">Copyright &copy;<script>document.write(new Date().getFullYear());</script> All rights reserved | This template is made by <a href="https://colorlib.com" target="_blank">Colorlib</a></p>
            <!-- Link back to Colorlib can't be removed. Template is licensed under CC BY 3.0. -->
						
					</div>
				</div>
			</footer>	
			<!-- End footer Area -->

			<script src="js/vendor/jquery-2.2.4.min.js"></script>
			<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
			<script src="js/vendor/bootstrap.min.js"></script>			
			<script type="text/javascript" src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBhOdIF3Y9382fqJYt5I_sswSrEw5eihAA"></script>
  			<script src="js/easing.min.js"></script>			
			<script src="js/hoverIntent.js"></script>
			<script src="js/superfish.min.js"></script>	
			<script src="js/jquery.ajaxchimp.min.js"></script>
			<script src="js/jquery.magnific-popup.min.js"></script>	
			<script src="js/owl.carousel.min.js"></script>			
			<script src="js/jquery.sticky.js"></script>
			<script src="js/jquery.nice-select.min.js"></script>	
			<script src="js/waypoints.min.js"></script>
			<script src="js/jquery.counterup.min.js"></script>					
			<script src="js/parallax.min.js"></script>	
			<script src="js/mail-script.js"></script>	
			<script src="js/main.js"></script>  	
		</body>
	</html>



